

<!DOCTYPE html>
<html lang="en">
  <!-- This file is autogenerated! Don't edit it! -->
  <head>
    <!-- Required meta tags always come first -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="x-ua-compatible" content="ie=edge">

    <!-- Bootstrap CSS -->
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.4/css/bootstrap.min.css">
    <style>
      body {
        position: relative;
        color: #111;
      }
      p.card-text{
        margin-bottom: 0.5rem;
      }
      h6.card-subtitle{
        margin-bottom: 0.5rem;
      }
    </style>
  </head>
  <body data-spy="scroll" data-target="#navlist">
    <div class="container">
      <div class="row" style="margin-top: 1rem;">
        <div class="col-xs-12 col-lg-9">
            <h5><a class="text-muted" href="index.html">
                <i class="fa fa-home" aria-hidden="true"></i>
                gitbib
            </a> |
            All tags:
            
            <span class="tag tag-default">algorithm</span>
            
            <span class="tag tag-default">cav</span>
            
            <span class="tag tag-default">cheminformatics</span>
            
            <span class="tag tag-default">deep-learning</span>
            
            <span class="tag tag-default">distributed-computing</span>
            
            <span class="tag tag-default">features</span>
            
            <span class="tag tag-default">forcefield</span>
            
            <span class="tag tag-default">kchan</span>
            
            <span class="tag tag-default">kv</span>
            
            <span class="tag tag-default">machine-learning</span>
            
            <span class="tag tag-default">md-algorithm</span>
            
            <span class="tag tag-default">md-applications</span>
            
            <span class="tag tag-default">md-sampling</span>
            
            <span class="tag tag-default">misc</span>
            
            <span class="tag tag-default">msm-applications</span>
            
            <span class="tag tag-default">msm-postprocessing</span>
            
            <span class="tag tag-default">msm-theory</span>
            
            <span class="tag tag-default">nav</span>
            
            <span class="tag tag-default">other-md-analysis</span>
            
            <span class="tag tag-default">perspective</span>
            
            <span class="tag tag-default">python</span>
            
            <span class="tag tag-default">qm</span>
            
            <span class="tag tag-default">review</span>
            
            <span class="tag tag-default">simulations</span>
            
            <span class="tag tag-default">software</span>
            
            <span class="tag tag-default">structure</span>
            
            <span class="tag tag-default">structures</span>
            
            <span class="tag tag-default">tica</span>
            
            <span class="tag tag-default">variational</span>
            
            </h5>
            
            
            <div class="card">
              <div class="card-block" id="n2017-deep-lift">
                <h4 class="card-title">Learning Important Features Through Propagating Activation Differences</h4>
                <h6 class="card-subtitle text-muted"><strong>2017-deep-lift</strong></h6>
                <p class="card-text">Avanti Shrikumar; Peyton Greenside; Anshul Kundaje</p>
                <p class="card-text">
                  
                  2017-04-10 (online)
                  
                  
                  
                </p>
                <p class="card-text">
                  
                  
                  
                  
                  
                  <a href="https://arxiv.org/abs/1704.02685">arxiv:1704.02685</a>
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Decompose ouput predictions</p>
<p class="card-text"></p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">machine-learning</span>
                  
                  <span class="tag tag-primary">deep-learning</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2017-herg-structure">
                <h4 class="card-title">Cryo-EM Structure of the Open Human  Ether-à-go-go -Related K +  Channel hERG</h4>
                <h6 class="card-subtitle text-muted"><strong>2017-herg-structure</strong></h6>
                <p class="card-text">Weiwei Wang; Roderick MacKinnon</p>
                <p class="card-text">
                  
                  
                  
                  2017-04-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Cell</em>.
                  169, 
                  3, 
                  422-430.e10. 
                  <a href="https://dx.doi.org/10.1016/j.cell.2017.03.048">doi:10.1016/j.cell.2017.03.048</a>
                  
                </p>
                

                
                <h6 class="m-b-0"><strong style="color: #444">PDB codes</strong></h6>
                <p class="card-text"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5va1">5va1</a>, <a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5va2">5va2</a>, <a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5va3">5va3</a></p>
                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">kv</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2017-navab">
                <h4 class="card-title">Structures of closed and open states of a voltage-gated sodium channel</h4>
                <h6 class="card-subtitle text-muted"><strong>2017-navab</strong></h6>
                <p class="card-text">Michael J. Lenaeus; Tamer M. Gamal El-Din; Christopher Ing; Karthik Ramanadane; Régis Pomès; Ning Zheng; William A. Catterall</p>
                <p class="card-text">
                  
                  2017-03-27 (online)
                  
                  
                  &ndash;
                  
                  
                  2017-04-11 (print)
                  
                </p>
                <p class="card-text">
                  <em>Proceedings of the National Academy of Sciences</em>.
                  114, 
                  15, 
                  E3051-E3060. 
                  <a href="https://dx.doi.org/10.1073/pnas.1700761114">doi:10.1073/pnas.1700761114</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Open PD/VSD (5vb8) and closed PD/VSD/CTD (5vb2)</p>
<p class="card-text"></p>
                

                
                <h6 class="m-b-0"><strong style="color: #444">PDB codes</strong></h6>
                <p class="card-text"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5vb8">5vb8</a>, <a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5vb2">5vb2</a></p>
                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">nav</span>
                  
                  <span class="tag tag-primary">structure</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2017-open-full-navms">
                <h4 class="card-title">The complete structure of an activated open sodium channel</h4>
                <h6 class="card-subtitle text-muted"><strong>2017-open-full-navms</strong></h6>
                <p class="card-text">Altin Sula; Jennifer Booker; Leo C. T. Ng; Claire E. Naylor; Paul G. DeCaen; B. A. Wallace</p>
                <p class="card-text">
                  
                  2017-02-16 (online)
                  
                  
                  
                </p>
                <p class="card-text">
                  <em>Nature Communications</em>.
                  8, 
                  
                  14205. 
                  <a href="https://dx.doi.org/10.1038/ncomms14205">doi:10.1038/ncomms14205</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">First full-length, open conformation. <a href="#n2017-navab">2017-navab</a> has a full-length closed.
This is NavMs and that is NavAb.</p>
<p class="card-text"></p>
                

                
                <h6 class="m-b-0"><strong style="color: #444">PDB codes</strong></h6>
                <p class="card-text"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5hvx">5hvx</a>, <a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5hvd">5hvd</a></p>
                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">nav</span>
                  
                  <span class="tag tag-primary">structure</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2017-open-navms">
                <h4 class="card-title">The complete structure of an activated open sodium channel</h4>
                <h6 class="card-subtitle text-muted"><strong>2017-open-navms</strong></h6>
                <p class="card-text">Altin Sula; Jennifer Booker; Leo C. T. Ng; Claire E. Naylor; Paul G. DeCaen; B. A. Wallace</p>
                <p class="card-text">
                  
                  2017-02-16 (online)
                  
                  
                  
                </p>
                <p class="card-text">
                  <em>Nature Communications</em>.
                  8, 
                  
                  14205. 
                  <a href="https://dx.doi.org/10.1038/ncomms14205">doi:10.1038/ncomms14205</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Open NavMs voltage gated sodium channel with all the parts (VSD + pore + CTD)</p>
<p class="card-text"></p>
                

                
                <h6 class="m-b-0"><strong style="color: #444">PDB codes</strong></h6>
                <p class="card-text"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5hvx">5hvx</a></p>
                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">nav</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2017-vsd-only-pmf">
                <h4 class="card-title">Simulating the Activation of Voltage Sensing Domain for a Voltage-Gated Sodium Channel Using Polarizable Force Field</h4>
                <h6 class="card-subtitle text-muted"><strong>2017-vsd-only-pmf</strong></h6>
                <p class="card-text">Rui-Ning Sun; Haipeng Gong</p>
                <p class="card-text">
                  
                  2017-02-09 (online)
                  
                  
                  &ndash;
                  
                  
                  2017-03-02 (print)
                  
                </p>
                <p class="card-text">
                  <em>The Journal of Physical Chemistry Letters</em>.
                  8, 
                  5, 
                  901-908. 
                  <a href="https://dx.doi.org/10.1021/acs.jpclett.7b00023">doi:10.1021/acs.jpclett.7b00023</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">They simulated only one domain. They took the NavAb vsd and connected it via molecular dynamics
to a homology model to a <a href="https://en.wikipedia.org/wiki/Ciona_intestinalis">sea squirt</a> vsd.
They used a polarizable force-field, which is more expensive than normal (fixed charge) forcefields
but that might be important for something so intertwined with moving electrical charges!</p>
<p class="card-text"></p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">nav</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2017-euk-navpas">
                <h4 class="card-title">Structure of a eukaryotic voltage-gated sodium channel at near-atomic resolution</h4>
                <h6 class="card-subtitle text-muted"><strong>2017-euk-navpas</strong></h6>
                <p class="card-text">Huaizong Shen; Qiang Zhou; Xiaojing Pan; Zhangqiang Li; Jianping Wu; Nieng Yan</p>
                <p class="card-text">
                  
                  2017-02-09 (online)
                  
                  
                  &ndash;
                  
                  
                  2017-03-03 (print)
                  
                </p>
                <p class="card-text">
                  <em>Science</em>.
                  355, 
                  6328, 
                  eaal4326. 
                  <a href="https://dx.doi.org/10.1126/science.aal4326">doi:10.1126/science.aal4326</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">First eukaryotic structure. CryoEM of cockroach NaV. Might not be a NaV.
No elecrophysiology can be performed. Was originally called PaFPC1, they renamed
it NaVPaS</p>
<p class="card-text"></p>
                

                
                <h6 class="m-b-0"><strong style="color: #444">PDB codes</strong></h6>
                <p class="card-text"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5x0m">5x0m</a></p>
                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">nav</span>
                  
                  <span class="tag tag-primary">structure</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2016-husic-msms">
                <h4 class="card-title">Optimized parameter selection reveals trends in Markov state models for protein folding</h4>
                <h6 class="card-subtitle text-muted"><strong>2016-husic-msms</strong></h6>
                <p class="card-text">Brooke E. Husic; Robert T. McGibbon; Mohammad M. Sultan; Vijay S. Pande</p>
                <p class="card-text">
                  
                  
                  
                  2016-11-21 (print)
                  
                </p>
                <p class="card-text">
                  <em>The Journal of Chemical Physics</em>.
                  145, 
                  19, 
                  194103. 
                  <a href="https://dx.doi.org/10.1063/1.4967809">doi:10.1063/1.4967809</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">The authors perform GMRQ cross validation on the twelve <a href="#n2011-larsen-folding">2011-larsen-folding</a> folding trajectories to
give guidelines for MSM construction.</p>
<p class="card-text">They present a flowchart for MSM construction that shows the three paths towards clustering: from
an rmsd distance metric, from features, or from tICA learned on features.</p>
<p class="card-text">They introduce GMRQ cross validation in the tradition of <a href="#n2015-mcgibbon-gmrq">2015-mcgibbon-gmrq (ref. 44)</a>.</p>
<p class="card-text">They present results but stress that you have to do your own cross validataion to be sure. Some conclusions
include: 1. tICA and PCA are better than direct clustering of features 2. when using tica, you can use kcenters,
kmeans, or minibatch kmeans to the same effect</p>
<p class="card-text">On one protein (2p6j) they look at all different features and show that they vary a lot. It's unfortunate that
this was only done on one protein.</p>
<p class="card-text"></p>
                

                

                
                    <a data-toggle="collapse" href="#n2016-husic-msms-cites"
                       aria-expanded="false" aria-controls="n2016-husic-msms-cites">
                    Show References
                    </a>
                    <table class="table table-sm collapse" id="n2016-husic-msms-cites">
                    <tr>
                    <th>Num</th><th>Entry</th><th>Why</th>
                    </tr>
                    
                        
                        <tr>
                        <td>41</td>
                        
                        <td><a href="#n2013-noe-variational">2013-noe-variational</a></td>
                        
                        <td>Variational principle</td>
                        </tr>
                        
                    
                        
                        <tr>
                        <td>42</td>
                        
                        <td><a href="#n2014-nuske-variational">2014-nuske-variational</a></td>
                        
                        <td>Variational principle</td>
                        </tr>
                        
                    
                        
                        <tr>
                        <td>5</td>
                        
                        <td><a href="#n2008-anton">2008-anton</a></td>
                        
                        <td>Generated the trajectories.</td>
                        </tr>
                        
                    
                        
                        <tr>
                        <td>18</td>
                        
                        <td><a href="#n2011-larsen-folding">2011-larsen-folding</a></td>
                        
                        <td>Re-analyzed these simulations. "Diversity of proteins analyzed"</td>
                        </tr>
                        
                    
                        
                        <tr>
                        <td>44</td>
                        
                        <td><a href="#n2015-mcgibbon-gmrq">2015-mcgibbon-gmrq</a></td>
                        
                        <td>Cross-validation</td>
                        </tr>
                        
                    
                    </table>
                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">msm-theory</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2016-commute-maps">
                <h4 class="card-title">Commute Maps: Separating Slowly Mixing Molecular Configurations for Kinetic Modeling</h4>
                <h6 class="card-subtitle text-muted"><strong>2016-commute-maps</strong></h6>
                <p class="card-text">Frank Noé; Ralf Banisch; Cecilia Clementi</p>
                <p class="card-text">
                  
                  
                  
                  2016-11-08 (print)
                  
                </p>
                <p class="card-text">
                  <em>Journal of Chemical Theory and Computation</em>.
                  12, 
                  11, 
                  5620-5630. 
                  <a href="https://dx.doi.org/10.1021/acs.jctc.6b00762">doi:10.1021/acs.jctc.6b00762</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Scale tIC coordinates by a function of the timescale.</p>
<p class="card-text"></p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">msm-theory</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2016-noe-reversible-tica">
                <h4 class="card-title">Variational approximation of molecular kinetics from short
  off-equilibrium simulations</h4>
                <h6 class="card-subtitle text-muted"><strong>2016-noe-reversible-tica</strong></h6>
                <p class="card-text">Hao Wu; Feliks Nüske; Fabian Paul; Stefan Klus; Peter Koltai; Frank Noé</p>
                <p class="card-text">
                  
                  2016-10-20 (online)
                  
                  
                  
                </p>
                <p class="card-text">
                  
                  
                  
                  
                  
                  <a href="https://arxiv.org/abs/1610.06773">arxiv:1610.06773</a>
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Provides a better way of "symetrizing" tICA correlation matrix. In tICA, you assume that the dynamics
are reversible. When we're learning from finite data, this reversibility isn't respected. Historically,
you take your correlation matrix, add its transpose, and divide by two. This is an especially poor
approximation if you have many short trajectories. This paper is analogous to the MLE method for
symetrizing MSM counts matrices.</p>
<p class="card-text"></p>
                

                

                
                    <a data-toggle="collapse" href="#n2016-noe-reversible-tica-cites"
                       aria-expanded="false" aria-controls="n2016-noe-reversible-tica-cites">
                    Show References
                    </a>
                    <table class="table table-sm collapse" id="n2016-noe-reversible-tica-cites">
                    <tr>
                    <th>Num</th><th>Entry</th><th>Why</th>
                    </tr>
                    
                        
                        <tr>
                        <td>46</td>
                        
                        <td><a href="#safe-css-doi10106314934536">doi:10.1063/1.4934536</a></td>
                        
                        <td>Reversibility makes analysis easier</td>
                        </tr>
                        
                    
                        
                        <tr>
                        <td>47</td>
                        
                        <td></td>
                        
                        <td>Simulations should be reversible</td>
                        </tr>
                        
                    
                        
                        <tr>
                        <td>22, 23, 28, 46, 48, 49</td>
                        <td></td>
                        <td>Reversible estimators were developed.</td>
                        </tr>
                        
                    
                        
                        <tr>
                        <td>50</td>
                        
                        <td><a href="#n2013-noe-variational">2013-noe-variational</a></td>
                        
                        <td></td>
                        </tr>
                        
                    
                        
                        <tr>
                        <td>51</td>
                        
                        <td><a href="#n2013-noe-tica">2013-noe-tica</a></td>
                        
                        <td></td>
                        </tr>
                        
                    
                        
                        <tr>
                        <td>52</td>
                        
                        <td><a href="#n2014-nuske-variational">2014-nuske-variational</a></td>
                        
                        <td></td>
                        </tr>
                        
                    
                        
                        <tr>
                        <td>50, 51, 52</td>
                        <td></td>
                        <td>Variational approach (VAC)</td>
                        </tr>
                        
                    
                    </table>
                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">msm-theory</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2016-neural-computers">
                <h4 class="card-title">Hybrid computing using a neural network with dynamic external memory</h4>
                <h6 class="card-subtitle text-muted"><strong>2016-neural-computers</strong></h6>
                <p class="card-text">Alex Graves; Greg Wayne; Malcolm Reynolds; Tim Harley; Ivo Danihelka; Agnieszka Grabska-Barwińska; Sergio Gómez Colmenarejo; Edward Grefenstette; Tiago Ramalho; John Agapiou; Adrià Puigdomènech Badia; Karl Moritz Hermann; Yori Zwols; Georg Ostrovski; Adam Cain; Helen King; Christopher Summerfield; Phil Blunsom; Koray Kavukcuoglu; Demis Hassabis</p>
                <p class="card-text">
                  
                  2016-10-12 (online)
                  
                  
                  
                </p>
                <p class="card-text">
                  <em>Nature</em>.
                  538, 
                  7626, 
                  471-476. 
                  <a href="https://dx.doi.org/10.1038/nature20101">doi:10.1038/nature20101</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Augment deep networks with an external memory (RAM) matrix.</p>
<p class="card-text">Bart says: "TL;DR: This work follows a line of research that teaches deep-nets
to learn algorithmic tasks (addition, sorting, multiplication, key-value look-up).
This paper goes a bit further and teaches their network to do shortest-path finding
in graphs and demonstrates on maps of the London underground. Cool demo with nice results,
but the hype-machine has blown it out of proportion
(check out the FT article for a breathless take claiming thinking computers are one step closer...)"</p>
<p class="card-text"></p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">machine-learning</span>
                  
                  <span class="tag tag-primary">deep-learning</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2016-aspuru-mol-feat">
                <h4 class="card-title">Automatic chemical design using a data-driven continuous representation
  of molecules</h4>
                <h6 class="card-subtitle text-muted"><strong>2016-aspuru-mol-feat</strong></h6>
                <p class="card-text">Rafael Gómez-Bombarelli; David Duvenaud; José Miguel Hernández-Lobato; Jorge Aguilera-Iparraguirre; Timothy D. Hirzel; Ryan P. Adams; Alán Aspuru-Guzik</p>
                <p class="card-text">
                  
                  2016-10-07 (online)
                  
                  
                  
                </p>
                <p class="card-text">
                  
                  
                  
                  
                  
                  <a href="https://arxiv.org/abs/1610.02415">arxiv:1610.02415</a>
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">The authors train an auto-encoder to provide a vector representation
for small molecules. Small molecules are graphs with varying
sizes, so they're hard to feed into neural nets (which require
fixed-length bitvectors). By fusing together an encoder and decoder
(and making the "middle" representation sufficiently small), they
learn a vector representation.</p>
<p class="card-text">The authors lean heavily on <a href="#safe-css-arxiv151106349">arxiv:1511.06349 (ref. 25)</a> to autoencode
SMILES strings.</p>
<p class="card-text">They use a variational autoencoder (noisy) to avoid "dead zones"
in latent space.</p>
<p class="card-text">They optomize OLED properties as an example.</p>
<p class="card-text"></p>
                

                

                
                    <a data-toggle="collapse" href="#n2016-aspuru-mol-feat-cites"
                       aria-expanded="false" aria-controls="n2016-aspuru-mol-feat-cites">
                    Show References
                    </a>
                    <table class="table table-sm collapse" id="n2016-aspuru-mol-feat-cites">
                    <tr>
                    <th>Num</th><th>Entry</th><th>Why</th>
                    </tr>
                    
                        
                        <tr>
                        <td>25</td>
                        
                        <td><a href="#safe-css-arxiv151106349">arxiv:1511.06349</a></td>
                        
                        <td></td>
                        </tr>
                        
                    
                    </table>
                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">machine-learning</span>
                  
                  <span class="tag tag-primary">cheminformatics</span>
                  
                  <span class="tag tag-primary">misc</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2016-msm-cryptic-binding">
                <h4 class="card-title">Modelling proteins’ hidden conformations to predict antibiotic resistance</h4>
                <h6 class="card-subtitle text-muted"><strong>2016-msm-cryptic-binding</strong></h6>
                <p class="card-text">Kathryn M. Hart; Chris M. W. Ho; Supratik Dutta; Michael L. Gross; Gregory R. Bowman</p>
                <p class="card-text">
                  
                  2016-10-06 (online)
                  
                  
                  
                </p>
                <p class="card-text">
                  <em>Nature Communications</em>.
                  7, 
                  
                  12965. 
                  <a href="https://dx.doi.org/10.1038/ncomms12965">doi:10.1038/ncomms12965</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Labmate summarizes:</p>
<p class="card-text">They generated ensembles using MD, then docked to those ensembles, then re-weighted the docking scores
based on the MSM. This gave a huge improvement in the predictive power of docking to predict affinity/potency.
It turned an inverse relationship (when docking using xtal structures) into a highly correlated trend.</p>
<p class="card-text">They confirmed their hypothesis about the protein flexibility by using a mass spec. method.</p>
<p class="card-text">They identified a loop movement important in the anti-antibacterial activity of the enzyme that was different
from one previously proposed/suspected.</p>
<p class="card-text">They proposed mutants that would stabilize their proposed loop, and tested them experimentally.</p>
<p class="card-text">The power of using the MSM to re-weight other analyses is also very encouraging
to see yet again. Also note that they did all this with what looks like a pretty low amount of aggregate
sampling (few microseconds per mutant).</p>
<p class="card-text"></p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">misc</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2016-mbar-volumes">
                <h4 class="card-title">Structural analysis of high-dimensional basins of attraction</h4>
                <h6 class="card-subtitle text-muted"><strong>2016-mbar-volumes</strong></h6>
                <p class="card-text">Stefano Martiniani; K. Julian Schrenk; Jacob D. Stevenson; David J. Wales; Daan Frenkel</p>
                <p class="card-text">
                  
                  2016-09-15 (online)
                  
                  
                  
                </p>
                <p class="card-text">
                  <em>Physical Review E</em>.
                  94, 
                  3, 
                  
                  <a href="https://dx.doi.org/10.1103/PhysRevE.94.031301">doi:10.1103/PhysRevE.94.031301</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Use multistate benett acceptance (MBAR) to find volumes in high dimensions.</p>
<p class="card-text"></p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">misc</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2016-guttenberg-deep-slow">
                <h4 class="card-title">Neural Coarse-Graining: Extracting slowly-varying latent degrees of
  freedom with neural networks</h4>
                <h6 class="card-subtitle text-muted"><strong>2016-guttenberg-deep-slow</strong></h6>
                <p class="card-text">Nicholas Guttenberg; Martin Biehl; Ryota Kanai</p>
                <p class="card-text">
                  
                  2016-09-01 (online)
                  
                  
                  
                </p>
                <p class="card-text">
                  
                  
                  
                  
                  
                  <a href="https://arxiv.org/abs/1609.00116">arxiv:1609.00116</a>
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Somehow uses deep networks to extract slow modes from dynamical signals.</p>
<p class="card-text"></p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">machine-learning</span>
                  
                  <span class="tag tag-primary">deep-learning</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2016-kratochvil-soft-knock">
                <h4 class="card-title">Instantaneous ion configurations in the K+ ion channel selectivity filter revealed by 2D IR spectroscopy</h4>
                <h6 class="card-subtitle text-muted"><strong>2016-kratochvil-soft-knock</strong></h6>
                <p class="card-text">H. T. Kratochvil; J. K. Carr; K. Matulef; A. W. Annen; H. Li; M. Maj; J. Ostmeyer; A. L. Serrano; H. Raghuraman; S. D. Moran; J. L. Skinner; E. Perozo; B. Roux; F. I. Valiyaveetil; M. T. Zanni</p>
                <p class="card-text">
                  
                  2016-09-01 (online)
                  
                  
                  &ndash;
                  
                  
                  2016-09-02 (print)
                  
                </p>
                <p class="card-text">
                  <em>Science</em>.
                  353, 
                  6303, 
                  1040-1044. 
                  <a href="https://dx.doi.org/10.1126/science.aag1447">doi:10.1126/science.aag1447</a>
                  
                </p>
                

                

                

                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2016-cav-structure">
                <h4 class="card-title">Structure of the voltage-gated calcium channel Cav1.1 at 3.6 Å resolution</h4>
                <h6 class="card-subtitle text-muted"><strong>2016-cav-structure</strong></h6>
                <p class="card-text">Jianping Wu; Zhen Yan; Zhangqiang Li; Xingyang Qian; Shan Lu; Mengqiu Dong; Qiang Zhou; Nieng Yan</p>
                <p class="card-text">
                  
                  2016-08-31 (online)
                  
                  
                  
                </p>
                <p class="card-text">
                  <em>Nature</em>.
                  537, 
                  7619, 
                  191-196. 
                  <a href="https://dx.doi.org/10.1038/nature19321">doi:10.1038/nature19321</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Cav structure. What's the diference between <a href="#n2015-cav-structure">2015-cav-structure</a></p>
                

                
                <h6 class="m-b-0"><strong style="color: #444">PDB codes</strong></h6>
                <p class="card-text"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5gjv">5gjv</a>, <a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5gjw">5gjw</a></p>
                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">cav</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2016-btx-insect">
                <h4 class="card-title">The Receptor Site and Mechanism of Action of Sodium Channel Blocker Insecticides</h4>
                <h6 class="card-subtitle text-muted"><strong>2016-btx-insect</strong></h6>
                <p class="card-text">Yongqiang Zhang; Yuzhe Du; Dingxin Jiang; Caitlyn Behnke; Yoshiko Nomura; Boris S. Zhorov; Ke Dong</p>
                <p class="card-text">
                  
                  2016-08-03 (online)
                  
                  
                  &ndash;
                  
                  
                  2016-09-16 (print)
                  
                </p>
                <p class="card-text">
                  <em>Journal of Biological Chemistry</em>.
                  291, 
                  38, 
                  20113-20124. 
                  <a href="https://dx.doi.org/10.1074/jbc.M116.742056">doi:10.1074/jbc.M116.742056</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Homology model to open cockroach channel and docking study</p>
<p class="card-text"></p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">nav</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2016-melcr-membrane-compare">
                <h4 class="card-title">Transmembrane Potential Modeling: Comparison between Methods of Constant Electric Field and Ion Imbalance</h4>
                <h6 class="card-subtitle text-muted"><strong>2016-melcr-membrane-compare</strong></h6>
                <p class="card-text">Josef Melcr; Daniel Bonhenry; Štěpán Timr; Pavel Jungwirth</p>
                <p class="card-text">
                  
                  
                  
                  2016-05-10 (print)
                  
                </p>
                <p class="card-text">
                  <em>J. Chem. Theory Comput. (Journal of Chemical Theory and Computation)</em>.
                  12, 
                  5, 
                  2418-2425. 
                  <a href="https://dx.doi.org/10.1021/acs.jctc.5b01202">doi:10.1021/acs.jctc.5b01202</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Compare external electric field with ion imbalance.</p>
<p class="card-text"></p>
                

                

                

                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2016-htmd">
                <h4 class="card-title">HTMD: High-Throughput Molecular Dynamics for Molecular Discovery</h4>
                <h6 class="card-subtitle text-muted"><strong>2016-htmd</strong></h6>
                <p class="card-text">S. Doerr; M. J. Harvey; Frank Noé; G. De Fabritiis</p>
                <p class="card-text">
                  
                  
                  
                  2016-04-12 (print)
                  
                </p>
                <p class="card-text">
                  <em>J. Chem. Theory Comput. (Journal of Chemical Theory and Computation)</em>.
                  12, 
                  4, 
                  1845-1852. 
                  <a href="https://dx.doi.org/10.1021/acs.jctc.6b00049">doi:10.1021/acs.jctc.6b00049</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">This can make MSMs in addition to being a one-stop shop for running MD.</p>
<p class="card-text">It's available under an academic license.</p>
<p class="card-text"></p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">software</span>
                  
                  <span class="tag tag-primary">python</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2016-sparsetica">
                <h4 class="card-title">Identification of simple reaction coordinates from complex dynamics</h4>
                <h6 class="card-subtitle text-muted"><strong>2016-sparsetica</strong></h6>
                <p class="card-text">Robert T. McGibbon; Vijay S. Pande</p>
                <p class="card-text">
                  
                  2016-02-28 (online)
                  
                  
                  
                </p>
                <p class="card-text">
                  
                  
                  
                  
                  
                  <a href="https://arxiv.org/abs/1602.08776">arxiv:1602.08776</a>
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">The authors argue for a definition of the reaction coordinate as the
projection on the dominant eigenfunciton of the propogator. Notably,
they say that path-based coordinates are no good, because progress
is only defined along the path. They argue that the coordinate shouldn't
depend on start and end points. They say the projection should
be maximally predictive. This means finding the slowest modes.
They note <a href="#n2006-nadler-diffusion-maps">2006-nadler-diffusion-maps (ref. 61)</a> and
<a href="#n2011-rohrdanz-diffusion-maps">2011-rohrdanz-diffusion-maps (ref. 62)</a> have used this definition.</p>
<p class="card-text">They go on to show tICA finds this reaction coordinate. To make tICA
more interpretable, they develop an algorithm for introducing a sparsity
pattern. It's a pseudo-l0 regularization (made smooth so the optimization works).</p>
<p class="card-text">They also use a unique dihedral featurization: instead of taking the sine and cosine
to get around periodicity concerns; they project the values on a bunch of evenly
spaced von-mises (periodic gaussians) distributions around the unit circle. Each
dihedral is expanded into several numbers. It's like a smooth histogramming.
This probably won't work as the number of dihedrals gets large (too many features).</p>
<p class="card-text"></p>
                

                

                
                    <a data-toggle="collapse" href="#n2016-sparsetica-cites"
                       aria-expanded="false" aria-controls="n2016-sparsetica-cites">
                    Show References
                    </a>
                    <table class="table table-sm collapse" id="n2016-sparsetica-cites">
                    <tr>
                    <th>Num</th><th>Entry</th><th>Why</th>
                    </tr>
                    
                        
                        <tr>
                        <td>61</td>
                        
                        <td><a href="#n2006-nadler-diffusion-maps">2006-nadler-diffusion-maps</a></td>
                        
                        <td></td>
                        </tr>
                        
                    
                        
                        <tr>
                        <td>62</td>
                        
                        <td><a href="#n2011-rohrdanz-diffusion-maps">2011-rohrdanz-diffusion-maps</a></td>
                        
                        <td></td>
                        </tr>
                        
                    
                    </table>
                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">msm-theory</span>
                  
                  <span class="tag tag-primary">tica</span>
                  
                  <span class="tag tag-primary">features</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2016-btx-frogs">
                <h4 class="card-title">Convergent Substitutions in a Sodium Channel Suggest Multiple Origins of Toxin Resistance in Poison Frogs</h4>
                <h6 class="card-subtitle text-muted"><strong>2016-btx-frogs</strong></h6>
                <p class="card-text">Rebecca D. Tarvin; Juan C. Santos; Lauren A. O'Connell; Harold H. Zakon; David C. Cannatella</p>
                <p class="card-text">
                  
                  2016-01-18 (online)
                  
                  
                  &ndash;
                  
                  
                  2016-04-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Molecular Biology and Evolution</em>.
                  33, 
                  4, 
                  1068-1081. 
                  <a href="https://dx.doi.org/10.1093/molbev/msv350">doi:10.1093/molbev/msv350</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Evolutionary analysis and docking study on NaV 1.4 in frogs immune to toxins</p>
<p class="card-text"></p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">nav</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2016-charmm-gui">
                <h4 class="card-title">CHARMM-GUI Input Generator for NAMD, GROMACS, AMBER, OpenMM, and CHARMM/OpenMM Simulations Using the CHARMM36 Additive Force Field</h4>
                <h6 class="card-subtitle text-muted"><strong>2016-charmm-gui</strong></h6>
                <p class="card-text">Jumin Lee; Xi Cheng; Jason M. Swails; Min Sun Yeom; Peter K. Eastman; Justin A. Lemkul; Shuai Wei; Joshua Buckner; Jong Cheol Jeong; Yifei Qi; Sunhwan Jo; Vijay S. Pande; David A. Case; Charles L. Brooks; Alexander D. MacKerell; Jeffery B. Klauda; Wonpil Im</p>
                <p class="card-text">
                  
                  
                  
                  2016-01-12 (print)
                  
                </p>
                <p class="card-text">
                  <em>J. Chem. Theory Comput. (Journal of Chemical Theory and Computation)</em>.
                  12, 
                  1, 
                  405-413. 
                  <a href="https://dx.doi.org/10.1021/acs.jctc.5b00935">doi:10.1021/acs.jctc.5b00935</a>
                  
                </p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">software</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2015-cav-structure">
                <h4 class="card-title">Structure of the voltage-gated calcium channel Cav1.1 complex</h4>
                <h6 class="card-subtitle text-muted"><strong>2015-cav-structure</strong></h6>
                <p class="card-text">J. Wu; Z. Yan; Z. Li; C. Yan; S. Lu; M. Dong; N. Yan</p>
                <p class="card-text">
                  
                  2015-12-17 (online)
                  
                  
                  &ndash;
                  
                  
                  2015-12-18 (print)
                  
                </p>
                <p class="card-text">
                  <em>Science</em>.
                  350, 
                  6267, 
                  aad2395-aad2395. 
                  <a href="https://dx.doi.org/10.1126/science.aad2395">doi:10.1126/science.aad2395</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">also a cav structure.</p>
                

                
                <h6 class="m-b-0"><strong style="color: #444">PDB codes</strong></h6>
                <p class="card-text"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3jbr">3jbr</a></p>
                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">cav</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="safe-css-arxiv151106349">
                <h4 class="card-title">Generating Sentences from a Continuous Space</h4>
                <h6 class="card-subtitle text-muted"><strong>arxiv:1511.06349</strong></h6>
                <p class="card-text">Samuel R. Bowman; Luke Vilnis; Oriol Vinyals; Andrew M. Dai; Rafal Jozefowicz; Samy Bengio</p>
                <p class="card-text">
                  
                  2015-11-19 (online)
                  
                  
                  
                </p>
                <p class="card-text">
                  
                  
                  
                  
                  
                  <a href="https://arxiv.org/abs/1511.06349">arxiv:1511.06349</a>
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Advances in autoencoding text, used by <a href="#n2016-aspuru-mol-feat">2016-aspuru-mol-feat</a>.
</p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">misc</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2015-pyemma">
                <h4 class="card-title">PyEMMA 2: A Software Package for Estimation, Validation, and Analysis of Markov Models</h4>
                <h6 class="card-subtitle text-muted"><strong>2015-pyemma</strong></h6>
                <p class="card-text">Martin K. Scherer; Benjamin Trendelkamp-Schroer; Fabian Paul; Guillermo Pérez-Hernández; Moritz Hoffmann; Nuria Plattner; Christoph Wehmeyer; Jan-Hendrik Prinz; Frank Noé</p>
                <p class="card-text">
                  
                  
                  
                  2015-11-10 (print)
                  
                </p>
                <p class="card-text">
                  <em>J. Chem. Theory Comput. (Journal of Chemical Theory and Computation)</em>.
                  11, 
                  11, 
                  5525-5542. 
                  <a href="https://dx.doi.org/10.1021/acs.jctc.5b00743">doi:10.1021/acs.jctc.5b00743</a>
                  
                </p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">software</span>
                  
                  <span class="tag tag-primary">python</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="safe-css-doi10106314934536">
                <h4 class="card-title">Estimation and uncertainty of reversible Markov models</h4>
                <h6 class="card-subtitle text-muted"><strong>doi:10.1063/1.4934536</strong></h6>
                <p class="card-text">Benjamin Trendelkamp-Schroer; Hao Wu; Fabian Paul; Frank Noé</p>
                <p class="card-text">
                  
                  
                  
                  2015-11-07 (print)
                  
                </p>
                <p class="card-text">
                  <em>The Journal of Chemical Physics (J. Chem. Phys.)</em>.
                  143, 
                  17, 
                  174101. 
                  <a href="https://dx.doi.org/10.1063/1.4934536">doi:10.1063/1.4934536</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Reversible estimates for MSMs</p>
                

                

                

                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2015-mdtraj">
                <h4 class="card-title">MDTraj: A Modern Open Library for the Analysis of Molecular Dynamics Trajectories</h4>
                <h6 class="card-subtitle text-muted"><strong>2015-mdtraj</strong></h6>
                <p class="card-text">Robert T. McGibbon; Kyle A. Beauchamp; Matthew P. Harrigan; Christoph Klein; Jason M. Swails; Carlos X. Hernández; Christian R. Schwantes; Lee-Ping Wang; Thomas J. Lane; Vijay S. Pande</p>
                <p class="card-text">
                  
                  
                  
                  2015-10-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Biophysical Journal</em>.
                  109, 
                  8, 
                  1528-1532. 
                  <a href="https://dx.doi.org/10.1016/j.bpj.2015.08.015">doi:10.1016/j.bpj.2015.08.015</a>
                  
                </p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">software</span>
                  
                  <span class="tag tag-primary">python</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2015-amino-acid-basis">
                <h4 class="card-title">A Basis Set for Peptides for the Variational Approach to Conformational Kinetics</h4>
                <h6 class="card-subtitle text-muted"><strong>2015-amino-acid-basis</strong></h6>
                <p class="card-text">F. Vitalini; F. Noé; B. G. Keller</p>
                <p class="card-text">
                  
                  
                  
                  2015-09-08 (print)
                  
                </p>
                <p class="card-text">
                  <em>Journal of Chemical Theory and Computation</em>.
                  11, 
                  9, 
                  3992-4004. 
                  <a href="https://dx.doi.org/10.1021/acs.jctc.5b00498">doi:10.1021/acs.jctc.5b00498</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Authors simulate individual (capped) amino acids for 1us / each and construct (mini-)MSMs on each one.
They use the outerproduct of these mini-MSMs to serve as a basis set for peptides. MiniMSMs are on
a grid in phi-psi angles. Since each miniMSM has approx 3 modes, the full basis would be 3^(N), which
is way too big! They call the second and third modes "excited states" and use a basis set that contains
a singly-exited residue. E.g. [11111] + [ [21111, 121111, 112111, 111211, ...] ].</p>
<p class="card-text">Alanine preceded by a proline is taken as a special case.</p>
<p class="card-text"></p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">msm-theory</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2015-gromacs">
                <h4 class="card-title">GROMACS: High performance molecular simulations through multi-level parallelism from laptops to supercomputers</h4>
                <h6 class="card-subtitle text-muted"><strong>2015-gromacs</strong></h6>
                <p class="card-text">Mark James Abraham; Teemu Murtola; Roland Schulz; Szilárd Páll; Jeremy C. Smith; Berk Hess; Erik Lindahl</p>
                <p class="card-text">
                  
                  
                  
                  2015-09-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>SoftwareX</em>.
                  1-2, 
                  
                  19-25. 
                  <a href="https://dx.doi.org/10.1016/j.softx.2015.06.001">doi:10.1016/j.softx.2015.06.001</a>
                  
                </p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">software</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2015-ratematrix">
                <h4 class="card-title">Efficient maximum likelihood parameterization of continuous-time Markov processes</h4>
                <h6 class="card-subtitle text-muted"><strong>2015-ratematrix</strong></h6>
                <p class="card-text">Robert T. McGibbon; Vijay S. Pande</p>
                <p class="card-text">
                  
                  
                  
                  2015-07-21 (print)
                  
                </p>
                <p class="card-text">
                  <em>The Journal of Chemical Physics (J. Chem. Phys.)</em>.
                  143, 
                  3, 
                  034109. 
                  <a href="https://dx.doi.org/10.1063/1.4926516">doi:10.1063/1.4926516</a>
                  
                </p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">msm-theory</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2015-mcgibbon-gmrq">
                <h4 class="card-title">Variational cross-validation of slow dynamical modes in molecular kinetics</h4>
                <h6 class="card-subtitle text-muted"><strong>2015-mcgibbon-gmrq</strong></h6>
                <p class="card-text">Robert T. McGibbon; Vijay S. Pande</p>
                <p class="card-text">
                  
                  
                  
                  2015-03-28 (print)
                  
                </p>
                <p class="card-text">
                  <em>The Journal of Chemical Physics</em>.
                  142, 
                  12, 
                  124105. 
                  <a href="https://dx.doi.org/10.1063/1.4916292">doi:10.1063/1.4916292</a>
                  
                </p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">msm-theory</span>
                  
                  <span class="tag tag-primary">variational</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2015-carpenter-structures">
                <h4 class="card-title">K2P channel gating mechanisms revealed by structures of TREK-2 and a complex with Prozac</h4>
                <h6 class="card-subtitle text-muted"><strong>2015-carpenter-structures</strong></h6>
                <p class="card-text">Y. Y. Dong; A. C. W. Pike; A. Mackenzie; C. McClenaghan; P. Aryal; L. Dong; A. Quigley; M. Grieben; S. Goubin; S. Mukhopadhyay; G. F. Ruda; M. V. Clausen; L. Cao; P. E. Brennan; N. A. Burgess-Brown; M. S. P. Sansom; S. J. Tucker; E. P. Carpenter</p>
                <p class="card-text">
                  
                  2015-03-12 (online)
                  
                  
                  &ndash;
                  
                  
                  2015-03-13 (print)
                  
                </p>
                <p class="card-text">
                  <em>Science</em>.
                  347, 
                  6227, 
                  1256-1259. 
                  <a href="https://dx.doi.org/10.1126/science.1261512">doi:10.1126/science.1261512</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Structures of up and down trek2.</p>
<p class="card-text">Cites <a href="#n2010-k2p-review">2010-k2p-review (ref. 1)</a> for background.
</p>
                

                
                <h6 class="m-b-0"><strong style="color: #444">PDB codes</strong></h6>
                <p class="card-text"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId={'code': '4XDJ', 'desc': 'down state'}">{'code': '4XDJ', 'desc': 'down state'}</a>, <a href="http://www.rcsb.org/pdb/explore/explore.do?structureId={'code': '4BW5', 'desc': 'up state'}">{'code': '4BW5', 'desc': 'up state'}</a>, <a href="http://www.rcsb.org/pdb/explore/explore.do?structureId={'code': '4XDL', 'desc': 'Br-fluoxetine complex, down'}">{'code': '4XDL', 'desc': 'Br-fluoxetine complex, down'}</a>, <a href="http://www.rcsb.org/pdb/explore/explore.do?structureId={'code': '4XDK', 'desc': 'norfluoxetine complex, down'}">{'code': '4XDK', 'desc': 'norfluoxetine complex, down'}</a></p>
                

                
                    <a data-toggle="collapse" href="#n2015-carpenter-structures-cites"
                       aria-expanded="false" aria-controls="n2015-carpenter-structures-cites">
                    Show References
                    </a>
                    <table class="table table-sm collapse" id="n2015-carpenter-structures-cites">
                    <tr>
                    <th>Num</th><th>Entry</th><th>Why</th>
                    </tr>
                    
                        
                        <tr>
                        <td>1</td>
                        
                        <td><a href="#n2010-k2p-review">2010-k2p-review</a></td>
                        
                        <td></td>
                        </tr>
                        
                    
                    </table>
                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">kchan</span>
                  
                  <span class="tag tag-primary">structures</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2015-gaussian-msms">
                <h4 class="card-title">Gaussian Markov transition models of molecular kinetics</h4>
                <h6 class="card-subtitle text-muted"><strong>2015-gaussian-msms</strong></h6>
                <p class="card-text">Hao Wu; Frank Noé</p>
                <p class="card-text">
                  
                  
                  
                  2015-02-28 (print)
                  
                </p>
                <p class="card-text">
                  <em>The Journal of Chemical Physics</em>.
                  142, 
                  8, 
                  084104. 
                  <a href="https://dx.doi.org/10.1063/1.4913214">doi:10.1063/1.4913214</a>
                  
                </p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">msm-theory</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2015-schwantes-ktica">
                <h4 class="card-title">Modeling Molecular Kinetics with tICA and the Kernel Trick</h4>
                <h6 class="card-subtitle text-muted"><strong>2015-schwantes-ktica</strong></h6>
                <p class="card-text">Christian R. Schwantes; Vijay S. Pande</p>
                <p class="card-text">
                  
                  
                  
                  2015-02-10 (print)
                  
                </p>
                <p class="card-text">
                  <em>J. Chem. Theory Comput. (Journal of Chemical Theory and Computation)</em>.
                  11, 
                  2, 
                  600-608. 
                  <a href="https://dx.doi.org/10.1021/ct5007357">doi:10.1021/ct5007357</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">They introduce kernel tICA as an extension to tICA. This is useful to get non-linear solutions to
the tICA equation. They claim you can estimate eigenprocesses without building an MSM.</p>
<p class="card-text">They briefly introduce the transfer operator. They introduce the variational principle of conformation
dynamics per <a href="#n2011-prinz">2011-prinz (ref. 25)</a>. They introduce tICA as maximizing the autocorrelation. They say that solutions to tICA
are the same as solutions to the variational problem per <a href="#n2013-noe-tica">2013-noe-tica (ref. 28)</a>. Linearity makes them crude solutions.</p>
<p class="card-text">They explain that a natural approach to introduce non-linearity is to expand the original representation
into a higher dimensional space and do tICA there. They say this is impractical. The expanded space
probably has to be huge. You can perform analysis in the big representation without explicitly representing it
by using the "kernel trick". They reproduce an example of the kernel trick from <a href="#n1998-scholkopf-kernel-pca">1998-scholkopf-kernel-pca (ref. 39)</a>.</p>
<p class="card-text">They re-write the tICA problem only in terms of inner products so you can apply the kernel trick. They introduce
normalization. They choose a gaussian kernel. They simulate a four-well potential, muller potential, alanine
dipeptide, and fip35ww. They need to do MLE cross validation over parameters (kernel width and regularization
strength).</p>
<p class="card-text">This uses so much RAM! Huge matrices to solve (that scale with the amount of data!!)</p>
<p class="card-text"></p>
                

                

                
                    <a data-toggle="collapse" href="#n2015-schwantes-ktica-cites"
                       aria-expanded="false" aria-controls="n2015-schwantes-ktica-cites">
                    Show References
                    </a>
                    <table class="table table-sm collapse" id="n2015-schwantes-ktica-cites">
                    <tr>
                    <th>Num</th><th>Entry</th><th>Why</th>
                    </tr>
                    
                        
                        <tr>
                        <td>21</td>
                        
                        <td><a href="#n2014-msm-perspective">2014-msm-perspective</a></td>
                        
                        <td>Data needs analysis</td>
                        </tr>
                        
                    
                        
                        <tr>
                        <td>25</td>
                        
                        <td><a href="#n2011-prinz">2011-prinz</a></td>
                        
                        <td></td>
                        </tr>
                        
                    
                        
                    
                        
                        <tr>
                        <td>25, 33</td>
                        <td></td>
                        <td>Details of transfer operator approach.</td>
                        </tr>
                        
                    
                        
                        <tr>
                        <td>34</td>
                        
                        <td><a href="#n2013-noe-variational">2013-noe-variational</a></td>
                        
                        <td>"It was shown that a variational principle can be derived for the eignvalues of the transfer operator."
The autocorelation of a function is less than the autocorrelation of the first dynamical eigenfunction
of the transfer operator. This is used to argue that you don't have to estimate the operator itself.
Just estimate its eigenfunctions
</td>
                        </tr>
                        
                    
                        
                        <tr>
                        <td>35</td>
                        
                        <td><a href="#n2014-nuske-variational">2014-nuske-variational</a></td>
                        
                        <td>"Successfully constructed estimates of the top eigenfunctions in the span of a prespecified library of basis functions."
Contrast with this work, which "does not require a predefined basis set"
</td>
                        </tr>
                        
                    
                        
                        <tr>
                        <td>22, 28, 36, 37, 38</td>
                        <td></td>
                        <td>Citing tICA</td>
                        </tr>
                        
                    
                        
                        <tr>
                        <td>22</td>
                        
                        <td><a href="#n2013-schwantes-tica">2013-schwantes-tica</a></td>
                        
                        <td></td>
                        </tr>
                        
                    
                        
                        <tr>
                        <td>28</td>
                        
                        <td><a href="#n2013-noe-tica">2013-noe-tica</a></td>
                        
                        <td>solutions to tica provide estimates of the slowest eigenfunctions of the transfer operator.</td>
                        </tr>
                        
                    
                        
                        <tr>
                        <td>36</td>
                        
                        <td><a href="#safe-css-doi101103PhysRevLett723634">doi:10.1103/PhysRevLett.72.3634</a></td>
                        
                        <td></td>
                        </tr>
                        
                    
                        
                        <tr>
                        <td>37</td>
                        
                        <td><a href="#safe-css-doi101162neco200618102495">doi:10.1162/neco.2006.18.10.2495</a></td>
                        
                        <td></td>
                        </tr>
                        
                    
                        
                        <tr>
                        <td>22, 28</td>
                        <td></td>
                        <td>tICA has been useful for finding slow timescales in simulations</td>
                        </tr>
                        
                    
                        
                        <tr>
                        <td>39</td>
                        
                        <td><a href="#n1998-scholkopf-kernel-pca">1998-scholkopf-kernel-pca</a></td>
                        
                        <td>Used to introduce ther kernel trick.</td>
                        </tr>
                        
                    
                        
                        <tr>
                        <td>39, 40, 41, 42, 43, 44</td>
                        <td></td>
                        <td>Kernels are used in machine learning.</td>
                        </tr>
                        
                    
                        
                        <tr>
                        <td>45, 46</td>
                        <td></td>
                        <td>Methods for simplifying kernel method computation.</td>
                        </tr>
                        
                    
                    </table>
                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">msm-theory</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2014-kopfer-hard-knock">
                <h4 class="card-title">Ion permeation in K+ channels occurs by direct Coulomb knock-on</h4>
                <h6 class="card-subtitle text-muted"><strong>2014-kopfer-hard-knock</strong></h6>
                <p class="card-text">D. A. Kopfer; C. Song; T. Gruene; G. M. Sheldrick; U. Zachariae; B. L. de Groot</p>
                <p class="card-text">
                  
                  2014-10-16 (online)
                  
                  
                  &ndash;
                  
                  
                  2014-10-17 (print)
                  
                </p>
                <p class="card-text">
                  <em>Science</em>.
                  346, 
                  6207, 
                  352-355. 
                  <a href="https://dx.doi.org/10.1126/science.1254840">doi:10.1126/science.1254840</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Introduces a new way of simulating a membrane potential: They
stack two membranes on top of one another, creating an "inside"
between the two. This doesn't hurt simulation throughput, because
you get twice as much protein motion data in the same amount of
simulation time (ignore extra factor of log n in system size).
This seems to be a refinement on their earlier work in
<a href="#n2011-kutzner-double-membrane">2011-kutzner-double-membrane (ref. 25)</a>.</p>
<p class="card-text">They hide the startling fact that every time an ion moves through
the channel, they have to instantaneously move it back inside.
Benoit has argued that this instantaneous jump, which can be
a 100 mV difference is rediculous.</p>
<p class="card-text">The main point of this paper is that ions translocate through
the four sites of potassium channel without any waters between
them. This "hard knock" mechanism is in contrast to a "soft knock"
mechanism where the ion-ion interactions are softened by interviening
waters.</p>
<p class="card-text">They re-refine the xray data to show it is consistent with the
hard-knock mechanism.</p>
<p class="card-text"></p>
                

                

                
                    <a data-toggle="collapse" href="#n2014-kopfer-hard-knock-cites"
                       aria-expanded="false" aria-controls="n2014-kopfer-hard-knock-cites">
                    Show References
                    </a>
                    <table class="table table-sm collapse" id="n2014-kopfer-hard-knock-cites">
                    <tr>
                    <th>Num</th><th>Entry</th><th>Why</th>
                    </tr>
                    
                        
                        <tr>
                        <td>25</td>
                        
                        <td><a href="#n2011-kutzner-double-membrane">2011-kutzner-double-membrane</a></td>
                        
                        <td></td>
                        </tr>
                        
                    
                    </table>
                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">kchan</span>
                  
                  <span class="tag tag-primary">md-applications</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2014-msm-perspective">
                <h4 class="card-title">Perspective: Markov models for long-timescale biomolecular dynamics</h4>
                <h6 class="card-subtitle text-muted"><strong>2014-msm-perspective</strong></h6>
                <p class="card-text">C. R. Schwantes; R. T. McGibbon; V. S. Pande</p>
                <p class="card-text">
                  
                  
                  
                  2014-09-07 (print)
                  
                </p>
                <p class="card-text">
                  <em>The Journal of Chemical Physics</em>.
                  141, 
                  9, 
                  090901. 
                  <a href="https://dx.doi.org/10.1063/1.4895044">doi:10.1063/1.4895044</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Very good perspective on the importance of analysis (particularly MSM
analysis) for understanding large, modern MD datasets. Money quote:
"we believe that quantitative analysis has increasingly become a
limiting factor in the application of MD"
</p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">msm-theory</span>
                  
                  <span class="tag tag-primary">perspective</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2014-ganguli-saddle-points">
                <h4 class="card-title">Identifying and attacking the saddle point problem in high-dimensional
  non-convex optimization</h4>
                <h6 class="card-subtitle text-muted"><strong>2014-ganguli-saddle-points</strong></h6>
                <p class="card-text">Yann Dauphin; Razvan Pascanu; Caglar Gulcehre; Kyunghyun Cho; Surya Ganguli; Yoshua Bengio</p>
                <p class="card-text">
                  
                  2014-06-10 (online)
                  
                  
                  
                </p>
                <p class="card-text">
                  
                  
                  
                  
                  
                  <a href="https://arxiv.org/abs/1406.2572">arxiv:1406.2572</a>
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Labmate summarizes:</p>
<p class="card-text">This one is a really cool paper.  One of those "we've all been doing it wrong" papers that could have a big
impact.  Their main conclusions are</p>
<p class="card-text">1. When optimizing functions in high dimensional spaces, saddle points are a much bigger problem than local
minima.  There are far more of them, and the few local minima that do exist mostly have values only slightly
worse than the global minimum.</p>
<p class="card-text">2. Standard optimization methods deal really badly with saddle points (and hence work really badly in high
dimensional spaces).  First order methods like gradient descent start taking tiny steps, so they take a really
long time to escape.  Quasi-Newton methods are even worse.  They just converge to the saddle point and never escape.</p>
<p class="card-text">3. They describe a new approach that doesn't have these problems and goes right through saddle points without
slowing down.</p>
<p class="card-text">They do all this in the context of neural networks, but it likely applies just as well to other high dimensional
optimization problems.  Proteins, for example.  When you use an algorithm like L-BFGS for energy minimization,
it's probably converging to a saddle point, not a local minimum.  It could be really interesting to try their
method.  Could we fold a protein to the native state just by a straightforward energy minimization?</p>
<p class="card-text">Force field optimization is another case whether this approach could be really useful.</p>
<p class="card-text">They also show that at a saddle point, there's a strong monotonic relationship between the error and the
fraction of negative eigenvalues of the Hessian.  Potentially that could be used as a way to measure how
far you are from the global minimum.  For example, when optimizing force field parameters, it would tell you
whether your parameters are close to optimal, or whether there's still a lot of room to improve them further.</p>
<p class="card-text"></p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">misc</span>
                  
                  <span class="tag tag-primary">machine-learning</span>
                  
                  <span class="tag tag-primary">deep-learning</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2014-nuske-variational">
                <h4 class="card-title">Variational Approach to Molecular Kinetics</h4>
                <h6 class="card-subtitle text-muted"><strong>2014-nuske-variational</strong></h6>
                <p class="card-text">Feliks Nüske; Bettina G. Keller; Guillermo Pérez-Hernández; Antonia S. J. S. Mey; Frank Noé</p>
                <p class="card-text">
                  
                  
                  
                  2014-04-08 (print)
                  
                </p>
                <p class="card-text">
                  <em>J. Chem. Theory Comput. (Journal of Chemical Theory and Computation)</em>.
                  10, 
                  4, 
                  1739-1752. 
                  <a href="https://dx.doi.org/10.1021/ct4009156">doi:10.1021/ct4009156</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">This paper is largely redundant with <a href="#n2013-noe-variational">2013-noe-variational (ref. 65)</a>. They cite it as such:
"Following the recently introduced variational principle for metastable stochastic processes,(65)
we propose a variational approach to molecular kinetics."</p>
<p class="card-text">They perform their variational approach on 2- and 10-alanine in addition to 1D potentials.</p>
<p class="card-text">This comes after tICA
and cites <a href="#n2013-schwantes-tica">2013-schwantes-tica (ref. 57)</a> and <a href="#n2013-noe-tica">2013-noe-tica (ref. 58)</a> in the intro, but does
nothing further with it. In particular, they don't note that tICA is just another
choice of basis set.</p>
<p class="card-text">They cite their error paper <a href="#n2010-msm-error">2010-msm-error (ref. 55)</a>.</p>
<p class="card-text"></p>
                

                

                
                    <a data-toggle="collapse" href="#n2014-nuske-variational-cites"
                       aria-expanded="false" aria-controls="n2014-nuske-variational-cites">
                    Show References
                    </a>
                    <table class="table table-sm collapse" id="n2014-nuske-variational-cites">
                    <tr>
                    <th>Num</th><th>Entry</th><th>Why</th>
                    </tr>
                    
                        
                        <tr>
                        <td>65</td>
                        
                        <td><a href="#n2013-noe-variational">2013-noe-variational</a></td>
                        
                        <td></td>
                        </tr>
                        
                    
                        
                        <tr>
                        <td>57</td>
                        
                        <td><a href="#n2013-schwantes-tica">2013-schwantes-tica</a></td>
                        
                        <td></td>
                        </tr>
                        
                    
                        
                        <tr>
                        <td>58</td>
                        
                        <td><a href="#n2013-noe-tica">2013-noe-tica</a></td>
                        
                        <td></td>
                        </tr>
                        
                    
                        
                        <tr>
                        <td>55</td>
                        
                        <td><a href="#n2010-msm-error">2010-msm-error</a></td>
                        
                        <td></td>
                        </tr>
                        
                    
                    </table>
                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">msm-theory</span>
                  
                  <span class="tag tag-primary">variational</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2014-chodera-msm">
                <h4 class="card-title">Markov state models of biomolecular conformational dynamics</h4>
                <h6 class="card-subtitle text-muted"><strong>2014-chodera-msm</strong></h6>
                <p class="card-text">John D Chodera; Frank Noé</p>
                <p class="card-text">
                  
                  
                  
                  2014-04-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Current Opinion in Structural Biology</em>.
                  25, 
                  
                  135-144. 
                  <a href="https://dx.doi.org/10.1016/j.sbi.2014.04.002">doi:10.1016/j.sbi.2014.04.002</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Overview of MSMs, stressing eigensystem and variational approach. Includes further reading suggestions.</p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">msm-theory</span>
                  
                  <span class="tag tag-primary">perspective</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2014-shukla-src-kinase">
                <h4 class="card-title">Activation pathway of Src kinase reveals intermediate states as targets for drug design</h4>
                <h6 class="card-subtitle text-muted"><strong>2014-shukla-src-kinase</strong></h6>
                <p class="card-text">Diwakar Shukla; Yilin Meng; Benoît Roux; Vijay S. Pande</p>
                <p class="card-text">
                  
                  2014-03-03 (online)
                  
                  
                  
                </p>
                <p class="card-text">
                  <em>Nature Communications</em>.
                  5, 
                  
                  
                  <a href="https://dx.doi.org/10.1038/ncomms4397">doi:10.1038/ncomms4397</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">MSM analysis of c-Src kinase. The MSMBuilder paper uses the dataset from this paper as an example.</p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">msm-applications</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2014-lipid14">
                <h4 class="card-title">Lipid14: The Amber Lipid Force Field</h4>
                <h6 class="card-subtitle text-muted"><strong>2014-lipid14</strong></h6>
                <p class="card-text">Callum J. Dickson; Benjamin D. Madej; Åge A. Skjevik; Robin M. Betz; Knut Teigen; Ian R. Gould; Ross C. Walker</p>
                <p class="card-text">
                  
                  
                  
                  2014-02-11 (print)
                  
                </p>
                <p class="card-text">
                  <em>J. Chem. Theory Comput. (Journal of Chemical Theory and Computation)</em>.
                  10, 
                  2, 
                  865-879. 
                  <a href="https://dx.doi.org/10.1021/ct4010307">doi:10.1021/ct4010307</a>
                  
                </p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">forcefield</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2014-closed-navae1p">
                <h4 class="card-title">Structure of a Prokaryotic Sodium Channel Pore Reveals Essential Gating Elements and an Outer Ion Binding Site Common to Eukaryotic Channels</h4>
                <h6 class="card-subtitle text-muted"><strong>2014-closed-navae1p</strong></h6>
                <p class="card-text">David Shaya; Felix Findeisen; Fayal Abderemane-Ali; Cristina Arrigoni; Stephanie Wong; Shailika Reddy Nurva; Gildas Loussouarn; Daniel L. Minor</p>
                <p class="card-text">
                  
                  
                  
                  2014-01-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Journal of Molecular Biology</em>.
                  426, 
                  2, 
                  467-483. 
                  <a href="https://dx.doi.org/10.1016/j.jmb.2013.10.010">doi:10.1016/j.jmb.2013.10.010</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Closed Pore and CTD in pore-only NavAe. Correlates CTD neck unfolding with activation.</p>
<p class="card-text"></p>
                

                
                <h6 class="m-b-0"><strong style="color: #444">PDB codes</strong></h6>
                <p class="card-text"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4lto">4lto</a>, <a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4ltp">4ltp</a>, <a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4ltq">4ltq</a>, <a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4ltr">4ltr</a></p>
                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">nav</span>
                  
                  <span class="tag tag-primary">structure</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2014-kohlhoff-exacycle">
                <h4 class="card-title">Cloud-based simulations on Google Exacycle reveal ligand modulation of GPCR activation pathways</h4>
                <h6 class="card-subtitle text-muted"><strong>2014-kohlhoff-exacycle</strong></h6>
                <p class="card-text">Kai J. Kohlhoff; Diwakar Shukla; Morgan Lawrenz; Gregory R. Bowman; David E. Konerding; Dan Belov; Russ B. Altman; Vijay S. Pande</p>
                <p class="card-text">
                  
                  2013-12-15 (online)
                  
                  
                  
                </p>
                <p class="card-text">
                  <em>Nature Chemistry</em>.
                  6, 
                  1, 
                  15-21. 
                  <a href="https://dx.doi.org/10.1038/nchem.1821">doi:10.1038/nchem.1821</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">They used Google's Exacycle to do these simulations. You can cite this for more examples of distributed computing. It's ostensibly about GPCRs.</p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">distributed-computing</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2013-open-pore-navms">
                <h4 class="card-title">Role of the C-terminal domain in the structure and function of tetrameric sodium channels</h4>
                <h6 class="card-subtitle text-muted"><strong>2013-open-pore-navms</strong></h6>
                <p class="card-text">Claire Bagnéris; Paul G. DeCaen; Benjamin A. Hall; Claire E. Naylor; David E. Clapham; Christopher W. M. Kay; B. A. Wallace</p>
                <p class="card-text">
                  
                  2013-09-19 (online)
                  
                  
                  
                </p>
                <p class="card-text">
                  <em>Nature Communications</em>.
                  4, 
                  
                  
                  <a href="https://dx.doi.org/10.1038/ncomms3465">doi:10.1038/ncomms3465</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Pore only open conformation. Supposed to have the CTD but
rcsb pdb doesn't show it. Proposes CTD role in gating.</p>
<p class="card-text"></p>
                

                
                <h6 class="m-b-0"><strong style="color: #444">PDB codes</strong></h6>
                <p class="card-text"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3zjz">3zjz</a></p>
                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">nav</span>
                  
                  <span class="tag tag-primary">structure</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2013-noe-variational">
                <h4 class="card-title">A Variational Approach to Modeling Slow Processes in Stochastic Dynamical Systems</h4>
                <h6 class="card-subtitle text-muted"><strong>2013-noe-variational</strong></h6>
                <p class="card-text">Frank Noé; Feliks Nüske</p>
                <p class="card-text">
                  
                  2013-06-26 (online)
                  
                  
                  
                </p>
                <p class="card-text">
                  <em>Multiscale Modeling & Simulation</em>.
                  11, 
                  2, 
                  635-655. 
                  <a href="https://dx.doi.org/10.1137/110858616">doi:10.1137/110858616</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">I think the point of this versus <a href="#n2014-nuske-variational">2014-nuske-variational</a> is to be "protein agnostic".
They allude to proteins, but say this is more general. Their example is a double-well
potential.</p>
<p class="card-text">They introduce the propogator formalism and stipulate that dynamics can be seperated
into "fast" and "slow" components. In contrast to a quantum mechanics Hamiltonian,
we don't know the propogator here. You have to infer it from data.</p>
<p class="card-text">They claim the error bound derived in <a href="#n2010-msm-error">2010-msm-error (ref. 34)</a> is not constructive, whereas this method *is*
constructive.</p>
<p class="card-text">Math section heavily cites <a href="#n2010-msm-error">2010-msm-error (ref. 34)</a>.</p>
<p class="card-text">They adapt the Rayleigh variational principle from quantum mechanics, and cite <a href="#n1989-szabo-ostlund-qm">1989-szabo-ostlund-qm (ref. 43)</a>.
They show that the autocorrelation of the true first dynamical eigenfunction is its
eigenvalue, and an estimate of the first dynamical eigenfunction necessarily has a
smaller eigenvalue. This sets the variational bound. In terms of names that don't seem
to be used now that we're in the future: the Ritz method is for when you have no overlap
integrals (e.g. MSMs) and the Roothan-Hall method is for when you do (tICA).</p>
<p class="card-text">They put it to the test on a double well potential. They use indicator basis functions to
make an MSM; hermite basis functions so they still have no overlap integrals, but smooth
functions; and gaussian basis functions (with overlap integrals). This must have
come before tICA because there is no mention made of it, even though it would fit in nicely.</p>
<p class="card-text"></p>
                

                

                
                    <a data-toggle="collapse" href="#n2013-noe-variational-cites"
                       aria-expanded="false" aria-controls="n2013-noe-variational-cites">
                    Show References
                    </a>
                    <table class="table table-sm collapse" id="n2013-noe-variational-cites">
                    <tr>
                    <th>Num</th><th>Entry</th><th>Why</th>
                    </tr>
                    
                        
                        <tr>
                        <td>34</td>
                        
                        <td><a href="#n2010-msm-error">2010-msm-error</a></td>
                        
                        <td></td>
                        </tr>
                        
                    
                        
                        <tr>
                        <td>34</td>
                        
                        <td><a href="#n2010-msm-error">2010-msm-error</a></td>
                        
                        <td></td>
                        </tr>
                        
                    
                        
                        <tr>
                        <td>43</td>
                        
                        <td><a href="#n1989-szabo-ostlund-qm">1989-szabo-ostlund-qm</a></td>
                        
                        <td></td>
                        </tr>
                        
                    
                    </table>
                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">msm-theory</span>
                  
                  <span class="tag tag-primary">variational</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2013-schwantes-tica">
                <h4 class="card-title">Improvements in Markov State Model Construction Reveal Many Non-Native Interactions in the Folding of NTL9</h4>
                <h6 class="card-subtitle text-muted"><strong>2013-schwantes-tica</strong></h6>
                <p class="card-text">Christian R. Schwantes; Vijay S. Pande</p>
                <p class="card-text">
                  
                  
                  
                  2013-04-09 (print)
                  
                </p>
                <p class="card-text">
                  <em>J. Chem. Theory Comput. (Journal of Chemical Theory and Computation)</em>.
                  9, 
                  4, 
                  2000-2009. 
                  <a href="https://dx.doi.org/10.1021/ct300878a">doi:10.1021/ct300878a</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">The Pande group introduces tica concomitantly with <a href="#n2013-noe-tica">2013-noe-tica</a>. This paper uses PCA as inspiration and cites signal processing literature.</p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">msm-theory</span>
                  
                  <span class="tag tag-primary">tica</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2013-milliseconds-folding">
                <h4 class="card-title">To milliseconds and beyond: challenges in the simulation of protein folding</h4>
                <h6 class="card-subtitle text-muted"><strong>2013-milliseconds-folding</strong></h6>
                <p class="card-text">Thomas J Lane; Diwakar Shukla; Kyle A Beauchamp; Vijay S Pande</p>
                <p class="card-text">
                  
                  
                  
                  2013-02-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Current Opinion in Structural Biology</em>.
                  23, 
                  1, 
                  58-65. 
                  <a href="https://dx.doi.org/10.1016/j.sbi.2012.11.002">doi:10.1016/j.sbi.2012.11.002</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">The state of folding simulations as it was in 2013. Has a nice plot of folding time by year by lab. Discusses the state of MSMs for analysis. Maybe cite this if you're doing folding or want to talk about how timescales are getting longer (and analysis is getting harder). The references include "recommended readings", which is nice.</p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">msm-theory</span>
                  
                  <span class="tag tag-primary">perspective</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2013-openmm">
                <h4 class="card-title">OpenMM 4: A Reusable, Extensible, Hardware Independent Library for High Performance Molecular Simulation</h4>
                <h6 class="card-subtitle text-muted"><strong>2013-openmm</strong></h6>
                <p class="card-text">Peter Eastman; Mark S. Friedrichs; John D. Chodera; Randall J. Radmer; Christopher M. Bruns; Joy P. Ku; Kyle A. Beauchamp; Thomas J. Lane; Lee-Ping Wang; Diwakar Shukla; Tony Tye; Mike Houston; Timo Stich; Christoph Klein; Michael R. Shirts; Vijay S. Pande</p>
                <p class="card-text">
                  
                  
                  
                  2013-01-08 (print)
                  
                </p>
                <p class="card-text">
                  <em>J. Chem. Theory Comput. (Journal of Chemical Theory and Computation)</em>.
                  9, 
                  1, 
                  461-469. 
                  <a href="https://dx.doi.org/10.1021/ct300857j">doi:10.1021/ct300857j</a>
                  
                </p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">software</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2013-noe-tica">
                <h4 class="card-title">Identification of slow molecular order parameters for Markov model construction</h4>
                <h6 class="card-subtitle text-muted"><strong>2013-noe-tica</strong></h6>
                <p class="card-text">Guillermo Pérez-Hernández; Fabian Paul; Toni Giorgino; Gianni De Fabritiis; Frank Noé</p>
                <p class="card-text">
                  
                  
                  
                  2013-01-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>The Journal of Chemical Physics (J. Chem. Phys.)</em>.
                  139, 
                  1, 
                  015102. 
                  <a href="https://dx.doi.org/10.1063/1.4811489">doi:10.1063/1.4811489</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">The Noe group introduces tica concomitantly with <a href="#n2013-schwantes-tica">2013-schwantes-tica</a>. They use the variational approach from <a href="#n2013-noe-variational">2013-noe-variational</a> to derive the tICA equation. They cite a 2001 book about independent component analysis.</p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">msm-theory</span>
                  
                  <span class="tag tag-primary">tica</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2012-partially-open-pore-navms">
                <h4 class="card-title">Structure of a bacterial voltage-gated sodium channel pore reveals mechanisms of opening and closing</h4>
                <h6 class="card-subtitle text-muted"><strong>2012-partially-open-pore-navms</strong></h6>
                <p class="card-text">Emily C. McCusker; Claire Bagnéris; Claire E. Naylor; Ambrose R. Cole; Nazzareno D'Avanzo; Colin G. Nichols; B.A. Wallace</p>
                <p class="card-text">
                  
                  2012-10-02 (online)
                  
                  
                  
                </p>
                <p class="card-text">
                  <em>Nature Communications</em>.
                  3, 
                  
                  1102. 
                  <a href="https://dx.doi.org/10.1038/ncomms2077">doi:10.1038/ncomms2077</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Pore only partially open state. This may or may not collapse to the
closed state in molecular dynamics.</p>
<p class="card-text"></p>
                

                
                <h6 class="m-b-0"><strong style="color: #444">PDB codes</strong></h6>
                <p class="card-text"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4f4l">4f4l</a></p>
                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">nav</span>
                  
                  <span class="tag tag-primary">structure</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2012-drid">
                <h4 class="card-title">Distribution of Reciprocal of Interatomic Distances: A Fast Structural Metric</h4>
                <h6 class="card-subtitle text-muted"><strong>2012-drid</strong></h6>
                <p class="card-text">Ting Zhou; Amedeo Caflisch</p>
                <p class="card-text">
                  
                  
                  
                  2012-08-14 (print)
                  
                </p>
                <p class="card-text">
                  <em>J. Chem. Theory Comput. (Journal of Chemical Theory and Computation)</em>.
                  8, 
                  8, 
                  2930-2937. 
                  <a href="https://dx.doi.org/10.1021/ct3003145">doi:10.1021/ct3003145</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">A unique featurization that encodes each atom by the first ~3 moments of its distribution of 1/distance to every other atom. Cite this if you use this featurization.</p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">features</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2012-jemma">
                <h4 class="card-title">EMMA: A Software Package for Markov Model Building and Analysis</h4>
                <h6 class="card-subtitle text-muted"><strong>2012-jemma</strong></h6>
                <p class="card-text">Martin Senne; Benjamin Trendelkamp-Schroer; Antonia S.J.S. Mey; Christof Schütte; Frank Noé</p>
                <p class="card-text">
                  
                  
                  
                  2012-07-10 (print)
                  
                </p>
                <p class="card-text">
                  <em>J. Chem. Theory Comput. (Journal of Chemical Theory and Computation)</em>.
                  8, 
                  7, 
                  2223-2238. 
                  <a href="https://dx.doi.org/10.1021/ct300274u">doi:10.1021/ct300274u</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">The previous, java version of EMMA. Look at <a href="#n2015-pyemma">2015-pyemma</a> instead.</p>
<p class="card-text"></p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">software</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2012-closed-navrh">
                <h4 class="card-title">Crystal structure of an orthologue of the NaChBac voltage-gated sodium channel</h4>
                <h6 class="card-subtitle text-muted"><strong>2012-closed-navrh</strong></h6>
                <p class="card-text">Xu Zhang; Wenlin Ren; Paul DeCaen; Chuangye Yan; Xiao Tao; Lin Tang; Jingjing Wang; Kazuya Hasegawa; Takashi Kumasaka; Jianhua He; Jiawei Wang; David E. Clapham; Nieng Yan</p>
                <p class="card-text">
                  
                  2012-05-20 (online)
                  
                  
                  
                </p>
                <p class="card-text">
                  <em>Nature</em>.
                  
                  
                  
                  <a href="https://dx.doi.org/10.1038/nature11054">doi:10.1038/nature11054</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Pore and Voltage Sensing domains in a closed state.</p>
<p class="card-text"></p>
                

                
                <h6 class="m-b-0"><strong style="color: #444">PDB codes</strong></h6>
                <p class="card-text"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4dxw">4dxw</a></p>
                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">nav</span>
                  
                  <span class="tag tag-primary">structure</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2012-closed-asym-navab">
                <h4 class="card-title">Crystal structure of a voltage-gated sodium channel in two potentially inactivated states</h4>
                <h6 class="card-subtitle text-muted"><strong>2012-closed-asym-navab</strong></h6>
                <p class="card-text">Jian Payandeh; Tamer M. Gamal El-Din; Todd Scheuer; Ning Zheng; William A. Catterall</p>
                <p class="card-text">
                  
                  2012-05-20 (online)
                  
                  
                  
                </p>
                <p class="card-text">
                  <em>Nature</em>.
                  
                  
                  
                  <a href="https://dx.doi.org/10.1038/nature11077">doi:10.1038/nature11077</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Pore and Voltage Sensing domains in an asymmetric closed state. NavAb.</p>
<p class="card-text"></p>
                

                
                <h6 class="m-b-0"><strong style="color: #444">PDB codes</strong></h6>
                <p class="card-text"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4ekw">4ekw</a></p>
                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">nav</span>
                  
                  <span class="tag tag-primary">structure</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2012-roux-efield">
                <h4 class="card-title">Constant electric field simulations of the membrane potential illustrated with simple systems</h4>
                <h6 class="card-subtitle text-muted"><strong>2012-roux-efield</strong></h6>
                <p class="card-text">James Gumbart; Fatemeh Khalili-Araghi; Marcos Sotomayor; Benoît Roux</p>
                <p class="card-text">
                  
                  
                  
                  2012-02-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Biochimica et Biophysica Acta (BBA) - Biomembranes</em>.
                  1818, 
                  2, 
                  294-302. 
                  <a href="https://dx.doi.org/10.1016/j.bbamem.2011.09.030">doi:10.1016/j.bbamem.2011.09.030</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Constant electric field</p>
<p class="card-text"></p>
                

                

                

                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2012-vsd-mechanism">
                <h4 class="card-title">Structural basis for gating charge movement in the voltage sensor of a sodium channel</h4>
                <h6 class="card-subtitle text-muted"><strong>2012-vsd-mechanism</strong></h6>
                <p class="card-text">V. Yarov-Yarovoy; P. G. DeCaen; R. E. Westenbroek; C.-Y. Pan; T. Scheuer; D. Baker; W. A. Catterall</p>
                <p class="card-text">
                  
                  2011-12-12 (online)
                  
                  
                  &ndash;
                  
                  
                  2012-01-10 (print)
                  
                </p>
                <p class="card-text">
                  <em>Proceedings of the National Academy of Sciences</em>.
                  109, 
                  2, 
                  E93-E102. 
                  <a href="https://dx.doi.org/10.1073/pnas.1118434109">doi:10.1073/pnas.1118434109</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Homology modelling to resting, intermediate and activated states of voltage sensing domains. Disulfide
locking experiments. Cites [doi:10.1073/pnas.0806486105] and [doi:10.1073/pnas.0912307106]
and [doi:10.1073/pnas.1116449108]</p>
<p class="card-text"></p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">nav</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2011-larsen-folding">
                <h4 class="card-title">How Fast-Folding Proteins Fold</h4>
                <h6 class="card-subtitle text-muted"><strong>2011-larsen-folding</strong></h6>
                <p class="card-text">K. Lindorff-Larsen; S. Piana; R. O. Dror; D. E. Shaw</p>
                <p class="card-text">
                  
                  2011-10-27 (online)
                  
                  
                  &ndash;
                  
                  
                  2011-10-28 (print)
                  
                </p>
                <p class="card-text">
                  <em>Science</em>.
                  334, 
                  6055, 
                  517-520. 
                  <a href="https://dx.doi.org/10.1126/science.1208351">doi:10.1126/science.1208351</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">The authors simulated folding trajectories for 12 small proteins. The simulations were
between 100 us and 1 ms. This paper was a considerable advance for the field, and more
or less closed the book on molecular dynamics for folding.
</p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">simulations</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2011-msmbuilder2">
                <h4 class="card-title">MSMBuilder2: Modeling Conformational Dynamics on the Picosecond to Millisecond Scale</h4>
                <h6 class="card-subtitle text-muted"><strong>2011-msmbuilder2</strong></h6>
                <p class="card-text">Kyle A. Beauchamp; Gregory R. Bowman; Thomas J. Lane; Lutz Maibaum; Imran S. Haque; Vijay S. Pande</p>
                <p class="card-text">
                  
                  
                  
                  2011-10-11 (print)
                  
                </p>
                <p class="card-text">
                  <em>J. Chem. Theory Comput. (Journal of Chemical Theory and Computation)</em>.
                  7, 
                  10, 
                  3412-3419. 
                  <a href="https://dx.doi.org/10.1021/ct200463m">doi:10.1021/ct200463m</a>
                  
                </p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">software</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2011-kutzner-double-membrane">
                <h4 class="card-title">Computational Electrophysiology: The Molecular Dynamics of Ion Channel Permeation and Selectivity in Atomistic Detail</h4>
                <h6 class="card-subtitle text-muted"><strong>2011-kutzner-double-membrane</strong></h6>
                <p class="card-text">Carsten Kutzner; Helmut Grubmüller; Bert L. de Groot; Ulrich Zachariae</p>
                <p class="card-text">
                  
                  
                  
                  2011-08-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Biophysical Journal</em>.
                  101, 
                  4, 
                  809-817. 
                  <a href="https://dx.doi.org/doi:10.1016/j.bpj.2011.06.010">doi:doi:10.1016/j.bpj.2011.06.010</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">They introduce the double-membrane scheme for measuring
ion conductance. They do it on a big ol' beta barrel.</p>
<p class="card-text"></p>
                

                

                

                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2011-closed-navab">
                <h4 class="card-title">The crystal structure of a voltage-gated sodium channel</h4>
                <h6 class="card-subtitle text-muted"><strong>2011-closed-navab</strong></h6>
                <p class="card-text">Jian Payandeh; Todd Scheuer; Ning Zheng; William A. Catterall</p>
                <p class="card-text">
                  
                  2011-07-10 (online)
                  
                  
                  
                </p>
                <p class="card-text">
                  <em>Nature</em>.
                  475, 
                  7356, 
                  353-358. 
                  <a href="https://dx.doi.org/10.1038/nature10238">doi:10.1038/nature10238</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Pore and Voltage Sensing domains in a closed state. First NaV Structure. Prokaryote NavAb</p>
<p class="card-text"></p>
                

                
                <h6 class="m-b-0"><strong style="color: #444">PDB codes</strong></h6>
                <p class="card-text"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3rvy">3rvy</a>, <a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3rvz">3rvz</a>, <a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3rw0">3rw0</a></p>
                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">nav</span>
                  
                  <span class="tag tag-primary">structure</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2011-kinase-review">
                <h4 class="card-title">Protein kinases: evolution of dynamic regulatory proteins</h4>
                <h6 class="card-subtitle text-muted"><strong>2011-kinase-review</strong></h6>
                <p class="card-text">Susan S. Taylor; Alexandr P. Kornev</p>
                <p class="card-text">
                  
                  
                  
                  2011-02-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Trends in Biochemical Sciences</em>.
                  36, 
                  2, 
                  65-77. 
                  <a href="https://dx.doi.org/10.1016/j.tibs.2010.09.006">doi:10.1016/j.tibs.2010.09.006</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Review of protein kinases. The MSMBuilder paper uses a kinase MD dataset as an example.</p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">review</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2011-rohrdanz-diffusion-maps">
                <h4 class="card-title">Determination of reaction coordinates via locally scaled diffusion map</h4>
                <h6 class="card-subtitle text-muted"><strong>2011-rohrdanz-diffusion-maps</strong></h6>
                <p class="card-text">Mary A. Rohrdanz; Wenwei Zheng; Mauro Maggioni; Cecilia Clementi</p>
                <p class="card-text">
                  
                  
                  
                  2011-01-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>The Journal of Chemical Physics (J. Chem. Phys.)</em>.
                  134, 
                  12, 
                  124116. 
                  <a href="https://dx.doi.org/10.1063/1.3569857">doi:10.1063/1.3569857</a>
                  
                </p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">other-md-analysis</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2011-prinz">
                <h4 class="card-title">Markov models of molecular kinetics: Generation and validation</h4>
                <h6 class="card-subtitle text-muted"><strong>2011-prinz</strong></h6>
                <p class="card-text">Jan-Hendrik Prinz; Hao Wu; Marco Sarich; Bettina Keller; Martin Senne; Martin Held; John D. Chodera; Christof Schütte; Frank Noé</p>
                <p class="card-text">
                  
                  
                  
                  2011-01-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>The Journal of Chemical Physics (J. Chem. Phys.)</em>.
                  134, 
                  17, 
                  174105. 
                  <a href="https://dx.doi.org/10.1063/1.3565032">doi:10.1063/1.3565032</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Fantastic in-depth intro to MSMs. Figure 1 in this paper is necessary for understanding
eigenvectors. This defines and relates the propogator and transfer operator. This shows
how we compute timescales from eigenvectors. This discusess state decomposition error
and shows that many states are needed in transition regions.
</p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">msm-theory</span>
                  
                  <span class="tag tag-primary">review</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2011-sklearn">
                <h4 class="card-title">Scikit-learn: Machine Learning in Python</h4>
                <h6 class="card-subtitle text-muted"><strong>2011-sklearn</strong></h6>
                <p class="card-text">F. Pedregosa; G. Varoquaux; A. Gramfort; V. Michel; B. Thirion; O. Grisel; M. Blondel; P. Prettenhofer; R. Weiss; V. Dubourg; J. Vanderplas; A. Passos; D. Cournapeau; M. Brucher; M. Perrot; E. Duchesnay</p>
                <p class="card-text">
                  
                  
                  
                  2011-01-01 (print)
                  
                </p>
                <p class="card-text">
                  
                  12, 
                  
                  2825-2830. 
                  
                  
                </p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">software</span>
                  
                  <span class="tag tag-primary">python</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2011-japan-tica">
                <h4 class="card-title">Slow dynamics in protein fluctuations revealed by time-structure based independent component analysis: The case of domain motions</h4>
                <h6 class="card-subtitle text-muted"><strong>2011-japan-tica</strong></h6>
                <p class="card-text">Yusuke Naritomi; Sotaro Fuchigami</p>
                <p class="card-text">
                  
                  
                  
                  2011-01-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>The Journal of Chemical Physics (J. Chem. Phys.)</em>.
                  134, 
                  6, 
                  065101. 
                  <a href="https://dx.doi.org/10.1063/1.3554380">doi:10.1063/1.3554380</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Probably the first application of tICA to MD.</p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">msm-theory</span>
                  
                  <span class="tag tag-primary">tica</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2010-shaw-fip35-bpti">
                <h4 class="card-title">Atomic-Level Characterization of the Structural Dynamics of Proteins</h4>
                <h6 class="card-subtitle text-muted"><strong>2010-shaw-fip35-bpti</strong></h6>
                <p class="card-text">D. E. Shaw; P. Maragakis; K. Lindorff-Larsen; S. Piana; R. O. Dror; M. P. Eastwood; J. A. Bank; J. M. Jumper; J. K. Salmon; Y. Shan; W. Wriggers</p>
                <p class="card-text">
                  
                  2010-10-14 (online)
                  
                  
                  &ndash;
                  
                  
                  2010-10-15 (print)
                  
                </p>
                <p class="card-text">
                  <em>Science</em>.
                  330, 
                  6002, 
                  341-346. 
                  <a href="https://dx.doi.org/10.1126/science.1187409">doi:10.1126/science.1187409</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Simulation of fip35 ww domain: 2x 100 us. Note this was at 400K so unfolding could be observed.</p>
<p class="card-text">Simulation of bpti: 1ms. Note this was done with tip4p for reasons.</p>
<p class="card-text"></p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">simulations</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2010-everything-msm-afraid-ask">
                <h4 class="card-title">Everything you wanted to know about Markov State Models but were afraid to ask</h4>
                <h6 class="card-subtitle text-muted"><strong>2010-everything-msm-afraid-ask</strong></h6>
                <p class="card-text">Vijay S. Pande; Kyle Beauchamp; Gregory R. Bowman</p>
                <p class="card-text">
                  
                  
                  
                  2010-09-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Methods</em>.
                  52, 
                  1, 
                  99-105. 
                  <a href="https://dx.doi.org/10.1016/j.ymeth.2010.06.002">doi:10.1016/j.ymeth.2010.06.002</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Review of MSMs intended for "non-experts". Obviously a little dated by now.</p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">msm-theory</span>
                  
                  <span class="tag tag-primary">review</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2010-k2p-review">
                <h4 class="card-title">Molecular Background of Leak K+ Currents: Two-Pore Domain Potassium Channels</h4>
                <h6 class="card-subtitle text-muted"><strong>2010-k2p-review</strong></h6>
                <p class="card-text">P. Enyedi; G. Czirjak</p>
                <p class="card-text">
                  
                  2010-04-14 (online)
                  
                  
                  &ndash;
                  
                  
                  2010-04-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Physiological Reviews</em>.
                  90, 
                  2, 
                  559-605. 
                  <a href="https://dx.doi.org/10.1152/physrev.00029.2009">doi:10.1152/physrev.00029.2009</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Nice review of K2P two-pore potassium channels. They talk
about the wide variety of regulatory stimuli</p>
<p class="card-text"></p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">kchan</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2010-gpugrid">
                <h4 class="card-title">High-Throughput All-Atom Molecular Dynamics Simulations Using Distributed Computing</h4>
                <h6 class="card-subtitle text-muted"><strong>2010-gpugrid</strong></h6>
                <p class="card-text">I. Buch; M. J. Harvey; T. Giorgino; D. P. Anderson; G. De Fabritiis</p>
                <p class="card-text">
                  
                  
                  
                  2010-03-22 (print)
                  
                </p>
                <p class="card-text">
                  <em>Journal of Chemical Information and Modeling (J. Chem. Inf. Model.)</em>.
                  50, 
                  3, 
                  397-403. 
                  <a href="https://dx.doi.org/10.1021/ci900455r">doi:10.1021/ci900455r</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">GPUGRID intro paper. Cite this alongside FAH. They (probably) did GPU distributed computing before FAH.</p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">distributed-computing</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2010-minibatch-kmeans">
                <h4 class="card-title">Web-scale k-means clustering</h4>
                <h6 class="card-subtitle text-muted"><strong>2010-minibatch-kmeans</strong></h6>
                <p class="card-text">D. Sculley</p>
                <p class="card-text">
                  
                  
                  
                  2010-01-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Proceedings of the 19th international conference on World wide web - WWW '10</em>.
                  
                  
                  
                  <a href="https://dx.doi.org/10.1145/1772690.1772862">doi:10.1145/1772690.1772862</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Clustering algorithm from sklearn admired for its speed.</p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">algorithm</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2010-msm-error">
                <h4 class="card-title">On the Approximation Quality of Markov State Models</h4>
                <h6 class="card-subtitle text-muted"><strong>2010-msm-error</strong></h6>
                <p class="card-text">Marco Sarich; Frank Noé; Christof Schütte</p>
                <p class="card-text">
                  
                  
                  
                  2010-01-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Multiscale Modeling & Simulation</em>.
                  8, 
                  4, 
                  1154-1177. 
                  <a href="https://dx.doi.org/10.1137/090764049">doi:10.1137/090764049</a>
                  
                </p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">msm-theory</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2009-noe-equilibrium-from-short">
                <h4 class="card-title">Constructing the equilibrium ensemble of folding pathways from short off-equilibrium simulations</h4>
                <h6 class="card-subtitle text-muted"><strong>2009-noe-equilibrium-from-short</strong></h6>
                <p class="card-text">F. Noe; C. Schutte; E. Vanden-Eijnden; L. Reich; T. R. Weikl</p>
                <p class="card-text">
                  
                  2009-11-03 (online)
                  
                  
                  &ndash;
                  
                  
                  2009-11-10 (print)
                  
                </p>
                <p class="card-text">
                  <em>Proceedings of the National Academy of Sciences</em>.
                  106, 
                  45, 
                  19011-19016. 
                  <a href="https://dx.doi.org/10.1073/pnas.0905466106">doi:10.1073/pnas.0905466106</a>
                  
                </p>
                

                

                

                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2009-msmbuilder1">
                <h4 class="card-title">Using generalized ensemble simulations and Markov state models to identify conformational states</h4>
                <h6 class="card-subtitle text-muted"><strong>2009-msmbuilder1</strong></h6>
                <p class="card-text">Gregory R. Bowman; Xuhui Huang; Vijay S. Pande</p>
                <p class="card-text">
                  
                  
                  
                  2009-10-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Methods</em>.
                  49, 
                  2, 
                  197-201. 
                  <a href="https://dx.doi.org/10.1016/j.ymeth.2009.04.013">doi:10.1016/j.ymeth.2009.04.013</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">This introduced the first release of MSMBuilder. You probably shouldn't cite this
unless you have a good reason to.</p>
<p class="card-text"></p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">software</span>
                  
                  <span class="tag tag-primary">python</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2009-friedrichs-gpu">
                <h4 class="card-title">Accelerating molecular dynamic simulation on graphics processing units</h4>
                <h6 class="card-subtitle text-muted"><strong>2009-friedrichs-gpu</strong></h6>
                <p class="card-text">Mark S. Friedrichs; Peter Eastman; Vishal Vaidyanathan; Mike Houston; Scott Legrand; Adam L. Beberg; Daniel L. Ensign; Christopher M. Bruns; Vijay S. Pande</p>
                <p class="card-text">
                  
                  
                  
                  2009-04-30 (print)
                  
                </p>
                <p class="card-text">
                  <em>Journal of Computational Chemistry</em>.
                  30, 
                  6, 
                  864-872. 
                  <a href="https://dx.doi.org/10.1002/jcc.21209">doi:10.1002/jcc.21209</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Probably the second instance of using GPUs for molecular dynamics. This became
<a href="http://openmm.org">OpenMM</a>.
</p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">md-sampling</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2009-theobald-rmsd">
                <h4 class="card-title">Fast determination of the optimal rotational matrix for macromolecular superpositions</h4>
                <h6 class="card-subtitle text-muted"><strong>2009-theobald-rmsd</strong></h6>
                <p class="card-text">Pu Liu; Dimitris K. Agrafiotis; Douglas L. Theobald</p>
                <p class="card-text">
                  
                  
                  
                  2009-01-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Journal of Computational Chemistry</em>.
                  
                  
                  n/a-n/a. 
                  <a href="https://dx.doi.org/10.1002/jcc.21439">doi:10.1002/jcc.21439</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">This one computes the optimal rotation in addition to (specifically: after) just
computing the minimal RMSD value. It uses <a href="#n2005-theobald-rmsd">2005-theobald-rmsd (ref. 14)</a> for finding the optimal RMSD (ie
leading eigenvalue of key matrix).
</p>
                

                

                
                    <a data-toggle="collapse" href="#n2009-theobald-rmsd-cites"
                       aria-expanded="false" aria-controls="n2009-theobald-rmsd-cites">
                    Show References
                    </a>
                    <table class="table table-sm collapse" id="n2009-theobald-rmsd-cites">
                    <tr>
                    <th>Num</th><th>Entry</th><th>Why</th>
                    </tr>
                    
                        
                        <tr>
                        <td>14</td>
                        
                        <td><a href="#n2005-theobald-rmsd">2005-theobald-rmsd</a></td>
                        
                        <td></td>
                        </tr>
                        
                    
                    </table>
                

                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2009-eastman-gpu">
                <h4 class="card-title">Efficient nonbonded interactions for molecular dynamics on a graphics processing unit</h4>
                <h6 class="card-subtitle text-muted"><strong>2009-eastman-gpu</strong></h6>
                <p class="card-text">Peter Eastman; Vijay S. Pande</p>
                <p class="card-text">
                  
                  
                  
                  2009-01-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Journal of Computational Chemistry</em>.
                  
                  
                  NA-NA. 
                  <a href="https://dx.doi.org/10.1002/jcc.21413">doi:10.1002/jcc.21413</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Optimizing below-cutoff nonbonded calculations on the GPU by tricky
memory and parallelization management. This was for OpenMM. This
is not PME.
</p>
                

                

                
                    <a data-toggle="collapse" href="#n2009-eastman-gpu-cites"
                       aria-expanded="false" aria-controls="n2009-eastman-gpu-cites">
                    Show References
                    </a>
                    <table class="table table-sm collapse" id="n2009-eastman-gpu-cites">
                    <tr>
                    <th>Num</th><th>Entry</th><th>Why</th>
                    </tr>
                    
                        
                    
                        
                    
                        
                    
                    </table>
                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">md-sampling</span>
                  
                  <span class="tag tag-primary">md-algorithm</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2009-metzner-tpt">
                <h4 class="card-title">Transition Path Theory for Markov Jump Processes</h4>
                <h6 class="card-subtitle text-muted"><strong>2009-metzner-tpt</strong></h6>
                <p class="card-text">Philipp Metzner; Christof Schütte; Eric Vanden-Eijnden</p>
                <p class="card-text">
                  
                  
                  
                  2009-01-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Multiscale Modeling & Simulation</em>.
                  7, 
                  3, 
                  1192-1219. 
                  <a href="https://dx.doi.org/10.1137/070699500">doi:10.1137/070699500</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Transition path theory (TPT).</p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">msm-theory</span>
                  
                  <span class="tag tag-primary">msm-postprocessing</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2009-berezhkovskii-reactive-flux">
                <h4 class="card-title">Reactive flux and folding pathways in network models of coarse-grained protein dynamics</h4>
                <h6 class="card-subtitle text-muted"><strong>2009-berezhkovskii-reactive-flux</strong></h6>
                <p class="card-text">Alexander Berezhkovskii; Gerhard Hummer; Attila Szabo</p>
                <p class="card-text">
                  
                  
                  
                  2009-01-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>The Journal of Chemical Physics (J. Chem. Phys.)</em>.
                  130, 
                  20, 
                  205102. 
                  <a href="https://dx.doi.org/10.1063/1.3139063">doi:10.1063/1.3139063</a>
                  
                </p>
                

                

                

                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2008-roux-efield">
                <h4 class="card-title">The Membrane Potential and its Representation by a Constant Electric Field in Computer Simulations</h4>
                <h6 class="card-subtitle text-muted"><strong>2008-roux-efield</strong></h6>
                <p class="card-text">Benoît Roux</p>
                <p class="card-text">
                  
                  
                  
                  2008-11-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Biophysical Journal</em>.
                  95, 
                  9, 
                  4205-4216. 
                  <a href="https://dx.doi.org/10.1529/biophysj.108.136499">doi:10.1529/biophysj.108.136499</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Constant electric field</p>
<p class="card-text"></p>
                

                

                

                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2008-anton">
                <h4 class="card-title">Anton, a special-purpose machine for molecular dynamics simulation</h4>
                <h6 class="card-subtitle text-muted"><strong>2008-anton</strong></h6>
                <p class="card-text">David E. Shaw; Jack C. Chao; Michael P. Eastwood; Joseph Gagliardo; J. P. Grossman; C. Richard Ho; Douglas J. Lerardi; István Kolossváry; John L. Klepeis; Timothy Layman; Christine McLeavey; Martin M. Deneroff; Mark A. Moraes; Rolf Mueller; Edward C. Priest; Yibing Shan; Jochen Spengler; Michael Theobald; Brian Towles; Stanley C. Wang; Ron O. Dror; Jeffrey S. Kuskin; Richard H. Larson; John K. Salmon; Cliff Young; Brannon Batson; Kevin J. Bowers</p>
                <p class="card-text">
                  
                  
                  
                  2008-07-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Communications of the ACM</em>.
                  51, 
                  7, 
                  91. 
                  <a href="https://dx.doi.org/10.1145/1364782.1364802">doi:10.1145/1364782.1364802</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">The seminal Anton paper. Cite this when talking about single, long
trajectories or special-purpose hardware.
</p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">md-sampling</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2008-anderson-gpu">
                <h4 class="card-title">General purpose molecular dynamics simulations fully implemented on graphics processing units</h4>
                <h6 class="card-subtitle text-muted"><strong>2008-anderson-gpu</strong></h6>
                <p class="card-text">Joshua A. Anderson; Chris D. Lorenz; A. Travesset</p>
                <p class="card-text">
                  
                  
                  
                  2008-05-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Journal of Computational Physics</em>.
                  227, 
                  10, 
                  5342-5359. 
                  <a href="https://dx.doi.org/10.1016/j.jcp.2008.01.047">doi:10.1016/j.jcp.2008.01.047</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">They claim to be the first GPU accelerated MD engine too! Probably led to HOOMD, although they
don't call it that in the paper.
</p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">md-sampling</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2007-stone-gpu">
                <h4 class="card-title">Accelerating molecular modeling applications with graphics processors</h4>
                <h6 class="card-subtitle text-muted"><strong>2007-stone-gpu</strong></h6>
                <p class="card-text">John E. Stone; James C. Phillips; Peter L. Freddolino; David J. Hardy; Leonardo G. Trabuco; Klaus Schulten</p>
                <p class="card-text">
                  
                  2007-01-01 (online)
                  
                  
                  &ndash;
                  
                  
                  2007-12-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Journal of Computational Chemistry</em>.
                  28, 
                  16, 
                  2618-2640. 
                  <a href="https://dx.doi.org/10.1002/jcc.20829">doi:10.1002/jcc.20829</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">(Probably) the first GPU accelerated MD paper. This is for NAMD.
</p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">md-sampling</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2007-ipython">
                <h4 class="card-title">IPython: A System for Interactive Scientific Computing</h4>
                <h6 class="card-subtitle text-muted"><strong>2007-ipython</strong></h6>
                <p class="card-text">Fernando Perez; Brian E. Granger</p>
                <p class="card-text">
                  
                  
                  
                  2007-01-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Computing in Science & Engineering</em>.
                  9, 
                  3, 
                  21-29. 
                  <a href="https://dx.doi.org/10.1109/MCSE.2007.53">doi:10.1109/MCSE.2007.53</a>
                  
                </p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">software</span>
                  
                  <span class="tag tag-primary">python</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="safe-css-doi101162neco200618102495">
                <h4 class="card-title">What Is the Relation Between Slow Feature Analysis and Independent Component Analysis?</h4>
                <h6 class="card-subtitle text-muted"><strong>doi:10.1162/neco.2006.18.10.2495</strong></h6>
                <p class="card-text">Tobias Blaschke; Pietro Berkes; Laurenz Wiskott</p>
                <p class="card-text">
                  
                  
                  
                  2006-10-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Neural Computation</em>.
                  18, 
                  10, 
                  2495-2508. 
                  <a href="https://dx.doi.org/10.1162/neco.2006.18.10.2495">doi:10.1162/neco.2006.18.10.2495</a>
                  
                </p>
                

                

                

                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2006-nadler-diffusion-maps">
                <h4 class="card-title">Diffusion maps, spectral clustering and reaction coordinates of dynamical systems</h4>
                <h6 class="card-subtitle text-muted"><strong>2006-nadler-diffusion-maps</strong></h6>
                <p class="card-text">Boaz Nadler; Stéphane Lafon; Ronald R. Coifman; Ioannis G. Kevrekidis</p>
                <p class="card-text">
                  
                  
                  
                  2006-07-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Applied and Computational Harmonic Analysis</em>.
                  21, 
                  1, 
                  113-127. 
                  <a href="https://dx.doi.org/10.1016/j.acha.2005.07.004">doi:10.1016/j.acha.2005.07.004</a>
                  
                </p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">other-md-analysis</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2005-theobald-rmsd">
                <h4 class="card-title">Rapid calculation of RMSDs using a quaternion-based characteristic polynomial</h4>
                <h6 class="card-subtitle text-muted"><strong>2005-theobald-rmsd</strong></h6>
                <p class="card-text">Douglas L. Theobald</p>
                <p class="card-text">
                  
                  2005-06-23 (online)
                  
                  
                  &ndash;
                  
                  
                  2005-07-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Acta Crystallographica Section A Foundations of Crystallography</em>.
                  61, 
                  4, 
                  478-480. 
                  <a href="https://dx.doi.org/10.1107/S0108767305015266">doi:10.1107/S0108767305015266</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Instead of doing matrix diagonalization or inversion, use netwon-raphson root-finding
on a characteristic polynomial.</p>
<p class="card-text">Mainly builds off of <a href="#n1987-horn-rmsd">1987-horn-rmsd (ref. 5)</a>.</p>
<p class="card-text"></p>
                

                

                
                    <a data-toggle="collapse" href="#n2005-theobald-rmsd-cites"
                       aria-expanded="false" aria-controls="n2005-theobald-rmsd-cites">
                    Show References
                    </a>
                    <table class="table table-sm collapse" id="n2005-theobald-rmsd-cites">
                    <tr>
                    <th>Num</th><th>Entry</th><th>Why</th>
                    </tr>
                    
                        
                        <tr>
                        <td>5</td>
                        
                        <td><a href="#n1987-horn-rmsd">1987-horn-rmsd</a></td>
                        
                        <td></td>
                        </tr>
                        
                    
                    </table>
                

                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2005-pcca">
                <h4 class="card-title">Robust Perron cluster analysis in conformation dynamics</h4>
                <h6 class="card-subtitle text-muted"><strong>2005-pcca</strong></h6>
                <p class="card-text">Peter Deuflhard; Marcus Weber</p>
                <p class="card-text">
                  
                  
                  
                  2005-03-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Linear Algebra and its Applications</em>.
                  398, 
                  
                  161-184. 
                  <a href="https://dx.doi.org/10.1016/j.laa.2004.10.026">doi:10.1016/j.laa.2004.10.026</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">PCCA group states based on an MSM transition matrix. Specifically,
it uses the eigenspectrum to do the lumping. Cite this in the methods
section of your paper if you use PCCA or PCCA+.
</p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">msm-theory</span>
                  
                  <span class="tag tag-primary">msm-postprocessing</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2004-swope-msm">
                <h4 class="card-title">Describing Protein Folding Kinetics by Molecular Dynamics Simulations. 1. Theory†</h4>
                <h6 class="card-subtitle text-muted"><strong>2004-swope-msm</strong></h6>
                <p class="card-text">William C. Swope; Jed W. Pitera; Frank Suits</p>
                <p class="card-text">
                  
                  
                  
                  2004-05-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>The Journal of Physical Chemistry B</em>.
                  108, 
                  21, 
                  6571-6581. 
                  <a href="https://dx.doi.org/10.1021/jp037421y">doi:10.1021/jp037421y</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">The first MSM paper. Gets pretty much everything right. Except they're convinced that you need
to do state exploration via NVT or NPT and then calculate transitions by launching bespoke
NVE simulations. Obviously, we just run big NPT runs and use that for both state space
exploration and counting transitions.
</p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">msm-theory</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2004-dill-rmsd">
                <h4 class="card-title">Using quaternions to calculate RMSD</h4>
                <h6 class="card-subtitle text-muted"><strong>2004-dill-rmsd</strong></h6>
                <p class="card-text">Evangelos A. Coutsias; Chaok Seok; Ken A. Dill</p>
                <p class="card-text">
                  
                  2004-01-01 (online)
                  
                  
                  &ndash;
                  
                  
                  2004-11-30 (print)
                  
                </p>
                <p class="card-text">
                  <em>Journal of Computational Chemistry</em>.
                  25, 
                  15, 
                  1849-1857. 
                  <a href="https://dx.doi.org/10.1002/jcc.20110">doi:10.1002/jcc.20110</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Similar to <a href="#n2005-theobald-rmsd">2005-theobald-rmsd</a>, builds off of <a href="#n1987-horn-rmsd">1987-horn-rmsd (ref. 5)</a>. Proves identity
with normal 3x3 methods.</p>
<p class="card-text">Derives the derivative of RMSD wrt coordinates, although "it is well known (ref. 13)"</p>
<p class="card-text"></p>
                

                

                
                    <a data-toggle="collapse" href="#n2004-dill-rmsd-cites"
                       aria-expanded="false" aria-controls="n2004-dill-rmsd-cites">
                    Show References
                    </a>
                    <table class="table table-sm collapse" id="n2004-dill-rmsd-cites">
                    <tr>
                    <th>Num</th><th>Entry</th><th>Why</th>
                    </tr>
                    
                        
                        <tr>
                        <td>5</td>
                        
                        <td><a href="#n1987-horn-rmsd">1987-horn-rmsd</a></td>
                        
                        <td></td>
                        </tr>
                        
                    
                    </table>
                

                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2001-schutte-variational">
                <h4 class="card-title">Transfer Operator Approach to Conformational Dynamics in Biomolecular Systems</h4>
                <h6 class="card-subtitle text-muted"><strong>2001-schutte-variational</strong></h6>
                <p class="card-text">Ch. Schütte; W. Huisinga; P. Deuflhard</p>
                <p class="card-text">
                  
                  
                  
                  2001-01-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Ergodic Theory, Analysis, and Efficient Simulation of Dynamical Systems</em>.
                  
                  
                  191-223. 
                  <a href="https://dx.doi.org/10.1007/978-3-642-56589-2_9">doi:10.1007/978-3-642-56589-2_9</a>
                  
                </p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">msm-theory</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2000-fah">
                <h4 class="card-title">COMPUTING: Screen Savers of the World Unite!</h4>
                <h6 class="card-subtitle text-muted"><strong>2000-fah</strong></h6>
                <p class="card-text">M. Shirts</p>
                <p class="card-text">
                  
                  
                  
                  2000-12-08 (print)
                  
                </p>
                <p class="card-text">
                  <em>Science</em>.
                  290, 
                  5498, 
                  1903-1904. 
                  <a href="https://dx.doi.org/10.1126/science.290.5498.1903">doi:10.1126/science.290.5498.1903</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">The seminal Folding at Home paper. Cite this whenever you talk about distributed computing
or Folding at Home.</p>
<p class="card-text">SETI@Home and distributed.net came before this.
</p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">distributed-computing</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n2000-roux-kcsa-md">
                <h4 class="card-title">Molecular Dynamics of the KcsA K+ Channel in a Bilayer Membrane</h4>
                <h6 class="card-subtitle text-muted"><strong>2000-roux-kcsa-md</strong></h6>
                <p class="card-text">Simon Bernèche; Benoît Roux</p>
                <p class="card-text">
                  
                  
                  
                  2000-06-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Biophysical Journal</em>.
                  78, 
                  6, 
                  2900-2917. 
                  <a href="https://dx.doi.org/10.1016/S0006-3495(00)76831-7">doi:10.1016/S0006-3495(00)76831-7</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">They run 4ns of MD on KcsA potassium channel.</p>
<p class="card-text"></p>
                

                

                

                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n1998-scholkopf-kernel-pca">
                <h4 class="card-title">Nonlinear Component Analysis as a Kernel Eigenvalue Problem</h4>
                <h6 class="card-subtitle text-muted"><strong>1998-scholkopf-kernel-pca</strong></h6>
                <p class="card-text">Bernhard Schölkopf; Alexander Smola; Klaus-Robert Müller</p>
                <p class="card-text">
                  
                  
                  
                  1998-07-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Neural Computation</em>.
                  10, 
                  5, 
                  1299-1319. 
                  <a href="https://dx.doi.org/10.1162/089976698300017467">doi:10.1162/089976698300017467</a>
                  
                </p>
                

                

                

                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n1996-vmd">
                <h4 class="card-title">VMD: Visual molecular dynamics</h4>
                <h6 class="card-subtitle text-muted"><strong>1996-vmd</strong></h6>
                <p class="card-text">William Humphrey; Andrew Dalke; Klaus Schulten</p>
                <p class="card-text">
                  
                  
                  
                  1996-02-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Journal of Molecular Graphics</em>.
                  14, 
                  1, 
                  33-38. 
                  <a href="https://dx.doi.org/10.1016/0263-7855(96)00018-5">doi:10.1016/0263-7855(96)00018-5</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">The only game in town for making movies.</p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">software</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n1995-stride">
                <h4 class="card-title">Knowledge-based protein secondary structure assignment</h4>
                <h6 class="card-subtitle text-muted"><strong>1995-stride</strong></h6>
                <p class="card-text">Dmitrij Frishman; Patrick Argos</p>
                <p class="card-text">
                  
                  
                  
                  1995-12-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Proteins: Structure, Function, and Genetics</em>.
                  23, 
                  4, 
                  566-579. 
                  <a href="https://dx.doi.org/10.1002/prot.340230412">doi:10.1002/prot.340230412</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">VMD wants you to cite this for secondary structure prediction</p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">software</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n1995-alpha-carbon">
                <h4 class="card-title">Cα-based torsion angles: A simple tool to analyze protein conformational changes</h4>
                <h6 class="card-subtitle text-muted"><strong>1995-alpha-carbon</strong></h6>
                <p class="card-text">Maria M. Flocco; Sherry L. Mowbray</p>
                <p class="card-text">
                  
                  
                  
                  1995-10-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Protein Science</em>.
                  4, 
                  10, 
                  2118-2122. 
                  <a href="https://dx.doi.org/10.1002/pro.5560041017">doi:10.1002/pro.5560041017</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Alpha carbon featurization</p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">features</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="safe-css-doi101103PhysRevLett723634">
                <h4 class="card-title">Separation of a mixture of independent signals using time delayed correlations</h4>
                <h6 class="card-subtitle text-muted"><strong>doi:10.1103/PhysRevLett.72.3634</strong></h6>
                <p class="card-text">L. Molgedey; H. G. Schuster</p>
                <p class="card-text">
                  
                  1994-06-06 (online)
                  
                  
                  
                </p>
                <p class="card-text">
                  <em>Phys. Rev. Lett. (Physical Review Letters)</em>.
                  72, 
                  23, 
                  3634-3637. 
                  <a href="https://dx.doi.org/10.1103/PhysRevLett.72.3634">doi:10.1103/PhysRevLett.72.3634</a>
                  
                </p>
                

                

                

                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n1992-wham">
                <h4 class="card-title">THE weighted histogram analysis method for free-energy calculations on biomolecules. I. The method</h4>
                <h6 class="card-subtitle text-muted"><strong>1992-wham</strong></h6>
                <p class="card-text">Shankar Kumar; John M. Rosenberg; Djamal Bouzida; Robert H. Swendsen; Peter A. Kollman</p>
                <p class="card-text">
                  
                  
                  
                  1992-10-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>J. Comput. Chem. (Journal of Computational Chemistry)</em>.
                  13, 
                  8, 
                  1011-1021. 
                  <a href="https://dx.doi.org/10.1002/jcc.540130812">doi:10.1002/jcc.540130812</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Wham reweighting algorithm, perhaps used after umbrella sampling.</p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">algorithm</span>
                  
                  <span class="tag tag-primary">md-algorithm</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n1989-szabo-ostlund-qm">
                <h4 class="card-title">Modern Quantum Chemistry: Introduction to Advanced Electronic Structure Theory</h4>
                <h6 class="card-subtitle text-muted"><strong>1989-szabo-ostlund-qm</strong></h6>
                <p class="card-text">Attila Szabo; Neil S. Ostlund</p>
                <p class="card-text">
                  
                  
                  
                  1989-01-01 (print)
                  
                </p>
                <p class="card-text">
                  
                  
                  
                  
                  
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text">Cited by <a href="#n2013-noe-variational">2013-noe-variational</a> for Rayleigh variational method.</p>
<p class="card-text"></p>
                

                

                

                
                <p class="card-text">
                  
                  <span class="tag tag-primary">qm</span>
                  
                </p>
                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n1987-horn-rmsd">
                <h4 class="card-title">Closed-form solution of absolute orientation using unit quaternions</h4>
                <h6 class="card-subtitle text-muted"><strong>1987-horn-rmsd</strong></h6>
                <p class="card-text">Berthold K. P. Horn</p>
                <p class="card-text">
                  
                  1987-04-01 (online)
                  
                  
                  &ndash;
                  
                  
                  1987-04-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Journal of the Optical Society of America A</em>.
                  4, 
                  4, 
                  629. 
                  <a href="https://dx.doi.org/10.1364/JOSAA.4.000629">doi:10.1364/JOSAA.4.000629</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text"><a href="#n2005-theobald-rmsd">2005-theobald-rmsd</a> cites for quaternion RMSD</p>
<p class="card-text"></p>
                

                

                

                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n1978-kabsch-rmsd">
                <h4 class="card-title">A solution for the best rotation to relate two sets of vectors</h4>
                <h6 class="card-subtitle text-muted"><strong>1978-kabsch-rmsd</strong></h6>
                <p class="card-text">W. Kabsch</p>
                <p class="card-text">
                  
                  
                  
                  1976-09-01 (print)
                  
                </p>
                <p class="card-text">
                  <em>Acta Crystallographica Section A</em>.
                  32, 
                  5, 
                  922-923. 
                  <a href="https://dx.doi.org/10.1107/S0567739476001873">doi:10.1107/S0567739476001873</a>
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text"><a href="#n2005-theobald-rmsd">2005-theobald-rmsd</a> says this suffers from rotoinversions because it uses
3x3 rotations instead of 4x4 quaternions.</p>
<p class="card-text"></p>
                

                

                

                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n1988-diamond-rmsd">
                <h4 class="card-title"></h4>
                <h6 class="card-subtitle text-muted"><strong>1988-diamond-rmsd</strong></h6>
                <p class="card-text"></p>
                <p class="card-text">
                  
                  
                  
                </p>
                <p class="card-text">
                  
                  
                  
                  
                  
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text"><a href="#n2005-theobald-rmsd">2005-theobald-rmsd</a> cites for quaternion RMSD</p>
<p class="card-text"></p>
                

                

                

                
              </div>
            </div>
            
            
            <div class="card">
              <div class="card-block" id="n1989-kearsley-rmsd">
                <h4 class="card-title"></h4>
                <h6 class="card-subtitle text-muted"><strong>1989-kearsley-rmsd</strong></h6>
                <p class="card-text"></p>
                <p class="card-text">
                  
                  
                  
                </p>
                <p class="card-text">
                  
                  
                  
                  
                  
                  
                </p>
                
                <h6 class="m-b-0"><strong style="color: #444">Description</strong></h6>
                <p class="card-text"><a href="#n2005-theobald-rmsd">2005-theobald-rmsd</a> cites for quaternion RMSD
</p>
                

                

                

                
              </div>
            </div>
            
        </div>
        <div class="col-xs-12 col-lg-3">
          <h4>Items</h4>
          <div id="navlist">
            <ul class="nav">
              
              
              <li class="nav-item">
                <a class="nav-link" href="#n2017-deep-lift">2017-deep-lift</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2017-herg-structure">2017-herg-structure</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2017-navab">2017-navab</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2017-open-full-navms">2017-open-full-navms</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2017-open-navms">2017-open-navms</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2017-vsd-only-pmf">2017-vsd-only-pmf</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2017-euk-navpas">2017-euk-navpas</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2016-husic-msms">2016-husic-msms</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2016-commute-maps">2016-commute-maps</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2016-noe-reversible-tica">2016-noe-reversible-tica</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2016-neural-computers">2016-neural-computers</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2016-aspuru-mol-feat">2016-aspuru-mol-feat</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2016-msm-cryptic-binding">2016-msm-cryptic-binding</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2016-mbar-volumes">2016-mbar-volumes</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2016-guttenberg-deep-slow">2016-guttenberg-deep-slow</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2016-kratochvil-soft-knock">2016-kratochvil-soft-knock</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2016-cav-structure">2016-cav-structure</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2016-btx-insect">2016-btx-insect</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2016-melcr-membrane-compare">2016-melcr-membrane-compare</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2016-htmd">2016-htmd</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2016-sparsetica">2016-sparsetica</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2016-btx-frogs">2016-btx-frogs</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2016-charmm-gui">2016-charmm-gui</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2015-cav-structure">2015-cav-structure</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#safe-css-arxiv151106349">arxiv:1511.06349</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2015-pyemma">2015-pyemma</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#safe-css-doi10106314934536">doi:10.1063/1.4934536</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2015-mdtraj">2015-mdtraj</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2015-amino-acid-basis">2015-amino-acid-basis</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2015-gromacs">2015-gromacs</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2015-ratematrix">2015-ratematrix</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2015-mcgibbon-gmrq">2015-mcgibbon-gmrq</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2015-carpenter-structures">2015-carpenter-structures</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2015-gaussian-msms">2015-gaussian-msms</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2015-schwantes-ktica">2015-schwantes-ktica</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2014-kopfer-hard-knock">2014-kopfer-hard-knock</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2014-msm-perspective">2014-msm-perspective</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2014-ganguli-saddle-points">2014-ganguli-saddle-points</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2014-nuske-variational">2014-nuske-variational</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2014-chodera-msm">2014-chodera-msm</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2014-shukla-src-kinase">2014-shukla-src-kinase</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2014-lipid14">2014-lipid14</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2014-closed-navae1p">2014-closed-navae1p</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2014-kohlhoff-exacycle">2014-kohlhoff-exacycle</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2013-open-pore-navms">2013-open-pore-navms</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2013-noe-variational">2013-noe-variational</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2013-schwantes-tica">2013-schwantes-tica</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2013-milliseconds-folding">2013-milliseconds-folding</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2013-openmm">2013-openmm</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2013-noe-tica">2013-noe-tica</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2012-partially-open-pore-navms">2012-partially-open-pore-navms</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2012-drid">2012-drid</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2012-jemma">2012-jemma</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2012-closed-navrh">2012-closed-navrh</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2012-closed-asym-navab">2012-closed-asym-navab</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2012-roux-efield">2012-roux-efield</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2012-vsd-mechanism">2012-vsd-mechanism</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2011-larsen-folding">2011-larsen-folding</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2011-msmbuilder2">2011-msmbuilder2</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2011-kutzner-double-membrane">2011-kutzner-double-membrane</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2011-closed-navab">2011-closed-navab</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2011-kinase-review">2011-kinase-review</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2011-rohrdanz-diffusion-maps">2011-rohrdanz-diffusion-maps</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2011-prinz">2011-prinz</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2011-sklearn">2011-sklearn</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2011-japan-tica">2011-japan-tica</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2010-shaw-fip35-bpti">2010-shaw-fip35-bpti</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2010-everything-msm-afraid-ask">2010-everything-msm-afraid-ask</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2010-k2p-review">2010-k2p-review</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2010-gpugrid">2010-gpugrid</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2010-minibatch-kmeans">2010-minibatch-kmeans</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2010-msm-error">2010-msm-error</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2009-noe-equilibrium-from-short">2009-noe-equilibrium-from-short</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2009-msmbuilder1">2009-msmbuilder1</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2009-friedrichs-gpu">2009-friedrichs-gpu</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2009-theobald-rmsd">2009-theobald-rmsd</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2009-eastman-gpu">2009-eastman-gpu</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2009-metzner-tpt">2009-metzner-tpt</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2009-berezhkovskii-reactive-flux">2009-berezhkovskii-reactive-flux</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2008-roux-efield">2008-roux-efield</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2008-anton">2008-anton</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2008-anderson-gpu">2008-anderson-gpu</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2007-stone-gpu">2007-stone-gpu</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2007-ipython">2007-ipython</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#safe-css-doi101162neco200618102495">doi:10.1162/neco.2006.18.10.2495</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2006-nadler-diffusion-maps">2006-nadler-diffusion-maps</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2005-theobald-rmsd">2005-theobald-rmsd</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2005-pcca">2005-pcca</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2004-swope-msm">2004-swope-msm</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2004-dill-rmsd">2004-dill-rmsd</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2001-schutte-variational">2001-schutte-variational</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2000-fah">2000-fah</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n2000-roux-kcsa-md">2000-roux-kcsa-md</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n1998-scholkopf-kernel-pca">1998-scholkopf-kernel-pca</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n1996-vmd">1996-vmd</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n1995-stride">1995-stride</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n1995-alpha-carbon">1995-alpha-carbon</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#safe-css-doi101103PhysRevLett723634">doi:10.1103/PhysRevLett.72.3634</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n1992-wham">1992-wham</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n1989-szabo-ostlund-qm">1989-szabo-ostlund-qm</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n1987-horn-rmsd">1987-horn-rmsd</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n1978-kabsch-rmsd">1978-kabsch-rmsd</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n1988-diamond-rmsd">1988-diamond-rmsd</a>
              </li>
            
              
              <li class="nav-item">
                <a class="nav-link" href="#n1989-kearsley-rmsd">1989-kearsley-rmsd</a>
              </li>
            
            </ul>
          </div>
        </div>
      </div>
    </div>

    <!-- jQuery first, then Tether, then Bootstrap JS. -->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.0.0/jquery.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.2.0/js/tether.min.js"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.4/js/bootstrap.min.js"></script>
    <script src="https://use.fontawesome.com/f678e12fb6.js"></script>
    <script>$('body').scrollspy({target: '#navlist'})</script>
  </body>
</html>